From 73aac2990df62f1470d5b983cba9ebbf5a4fb6f5 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 3 Dec 2025 18:32:21 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 fortran/src/CMakeLists.txt | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/fortran/src/CMakeLists.txt b/fortran/src/CMakeLists.txt
index d84143b00f0..fe576f7a7ae 100644
--- a/fortran/src/CMakeLists.txt
+++ b/fortran/src/CMakeLists.txt
@@ -151,14 +151,14 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5f90i_gen.h ${HDF5_F90_BINARY_DIR}/H5fortran_types.F90
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E env ${CROSSCOMPILING_PATH}
-        COMMAND    $<TARGET_FILE:H5match_types>
+        COMMAND    H5match_types
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5fortran_types.F90 and H5f90i_gen.h files"
     )
   else ()
     add_custom_command (TARGET H5match_types POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5f90i_gen.h ${HDF5_F90_BINARY_DIR}/H5fortran_types.F90
-        COMMAND    $<TARGET_FILE:H5match_types>
+        COMMAND    H5match_types
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5fortran_types.F90 and H5f90i_gen.h files"
     )
@@ -343,14 +343,14 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5_gen.F90")
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5_gen.F90
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E env ${CROSSCOMPILING_PATH}
-        COMMAND    $<TARGET_FILE:H5_buildiface>
+        COMMAND    H5_buildiface
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5_gen.F90 file"
     )
   else ()
     add_custom_command (TARGET H5_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5_gen.F90
-        COMMAND    $<TARGET_FILE:H5_buildiface>
+        COMMAND    H5_buildiface
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5_gen.F90 file"
     )
