From 673f419a1a738815837f9354b221193facc872d4 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Wed, 3 Dec 2025 12:07:17 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 fortran/src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fortran/src/CMakeLists.txt b/fortran/src/CMakeLists.txt
index d84143b00f0..e822a5c8d34 100644
--- a/fortran/src/CMakeLists.txt
+++ b/fortran/src/CMakeLists.txt
@@ -150,7 +150,7 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
     add_custom_command (TARGET H5match_types POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5f90i_gen.h ${HDF5_F90_BINARY_DIR}/H5fortran_types.F90
         COMMAND    ${CMAKE_COMMAND}
-        ARGS       -E env ${CROSSCOMPILING_PATH}
+
         COMMAND    $<TARGET_FILE:H5match_types>
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5fortran_types.F90 and H5f90i_gen.h files"
@@ -342,7 +342,7 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5_gen.F90")
     add_custom_command (TARGET H5_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5_gen.F90
         COMMAND    ${CMAKE_COMMAND}
-        ARGS       -E env ${CROSSCOMPILING_PATH}
+
         COMMAND    $<TARGET_FILE:H5_buildiface>
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5_gen.F90 file"
--- a/fortran/test/CMakeLists.txt
+++ b/fortran/test/CMakeLists.txt
@@ -108,7 +108,7 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/tf_gen.F90")
     add_custom_command (TARGET H5_test_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_FORTRAN_TESTS_BINARY_DIR}/tf_gen.F90
         COMMAND    ${CMAKE_COMMAND}
-        ARGS       -E env ${CROSSCOMPILING_PATH}
+
         COMMAND    $<TARGET_FILE:H5_test_buildiface>
         WORKING_DIRECTORY ${HDF5_FORTRAN_TESTS_BINARY_DIR}
         COMMENT "Generating the tf_gen.F90 static file"
