From 069bcde0b3346ef2e25a1372114bbce6467ca563 Mon Sep 17 00:00:00 2001
From: Thomas Staudinger <Staudi.Kaos@gmail.com>
Date: Thu, 26 Dec 2024 00:59:44 +0100
Subject: [PATCH] Specify version metadata as dynamic

Fixes build with maturin 1.8.0

Signed-off-by: Thomas Staudinger <Staudi.Kaos@gmail.com>
---
 pyproject.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyproject.toml b/pyproject.toml
index 56e82030..2134b966 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,6 @@
 [project]
 name = "orjson"
+dynamic = ["version"]
 repository = "https://github.com/ijl/orjson"
 requires-python = ">=3.8"
 classifiers = [
