From 28161505a04585780d54e7aa8a01c30517c41170 Mon Sep 17 00:00:00 2001
From: Michael Cho <michael@michaelcho.dev>
Date: Sun, 10 Aug 2025 21:35:05 -0400
Subject: [PATCH] Fix build with Boost 1.89.0

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index af87507..d2768c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,7 +63,10 @@ else()
 endif()
 set(Boost_USE_STATIC_RUNTIME OFF)
 
-find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
+find_package(Boost 1.47.0 COMPONENTS chrono REQUIRED)
+if(Boost_MAJOR_VERSION EQUAL 1 AND Boost_MINOR_VERSION LESS 69)
+  find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
+endif()
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 
