From d271adb8a795e6aa98c09507b3401fc4d8ab21c1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Liru=20F=C3=A6rs?= <liru@cubbi.com>
Date: Wed, 15 Jan 2025 18:26:02 -0500
Subject: [PATCH] Allow the build environment to override CMAKE_CXX_STANDARD

---
 CMakeLists.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7fae33c..af87507 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,9 @@ cmake_minimum_required(VERSION 3.5)
 
 project(SimpleAmqpClient LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 98)
+if(NOT DEFINED CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 98)
+endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 

From 28161505a04585780d54e7aa8a01c30517c41170 Mon Sep 17 00:00:00 2001
From: Michael Cho <michael@michaelcho.dev>
Date: Sun, 10 Aug 2025 21:35:05 -0400
Subject: [PATCH] Fix build with Boost 1.89.0

---
 CMakeLists.txt | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index af87507..d2768c7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -63,7 +63,10 @@ else()
 endif()
 set(Boost_USE_STATIC_RUNTIME OFF)
 
-find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
+find_package(Boost 1.47.0 COMPONENTS chrono REQUIRED)
+if(Boost_MAJOR_VERSION EQUAL 1 AND Boost_MINOR_VERSION LESS 69)
+  find_package(Boost 1.47.0 COMPONENTS chrono system REQUIRED)
+endif()
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 link_directories(${Boost_LIBRARY_DIRS})
 
