--- a/SGSNGGSN/Sgsn.cpp
+++ b/SGSNGGSN/Sgsn.cpp
@@ -13,6 +13,9 @@
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
 
+#include <ostream>
+#include <iostream>
+#include <sstream>
 #include <list>
 #if RN_UMTS
 #include <SIPInterface.h>
--- a/GPRS/MSInfo.cpp
+++ b/GPRS/MSInfo.cpp
@@ -13,6 +13,9 @@
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
 
+#include <ostream>
+#include <iostream>
+#include <sstream>
 #define LOG_GROUP LogGroup::GPRS		// Can set Log.Level.GPRS for debugging
 
 #include "MSInfo.h"
--- a/Control/L3StateMachine.cpp
+++ b/Control/L3StateMachine.cpp
@@ -15,6 +15,9 @@
 
 */
 // Written by Pat Thompson
+#include <ostream>
+#include <iostream>
+#include <sstream>
 #define LOG_GROUP LogGroup::Control		// Can set Log.Level.Control for debugging
 #include "L3StateMachine.h"
 #include "L3CallControl.h"
