--- a/SGSNGGSN/Sgsn.cpp
+++ b/SGSNGGSN/Sgsn.cpp
@@ -13,6 +13,7 @@
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 */
 
+#include <sstream>
 #include <list>
 #if RN_UMTS
 #include <SIPInterface.h>
--- a/GPRS/MSInfo.cpp
+++ b/GPRS/MSInfo.cpp
@@ -24,6 +24,7 @@
 #include "Sgsn.h"
 #include <string.h>		// for strchr
 #include <iomanip>		// For fmtflags, setprecision
+#include <sstream>
 
 namespace GPRS {
 
--- a/Control/L3StateMachine.cpp
+++ b/Control/L3StateMachine.cpp
@@ -32,6 +32,7 @@
 #include <RRLPServer.h>
 #include <Globals.h>
 #include <typeinfo>
+#include <sstream>
 
 using namespace GSM;
 namespace Control {
