--- a/SIP/SIP2Interface.cpp
+++ b/SIP/SIP2Interface.cpp
@@ -756,7 +756,7 @@ void MySipInterface::msiInit()
 	// FIXME -- Can we coordinate this with the global logger?
 	//ortp_set_log_level_mask(ORTP_MESSAGE|ORTP_WARNING|ORTP_ERROR);
 
-	ortp_set_log_handler(ortpLogFunc);
+	ortp_set_log_handler((BctbxLogFunc) ortpLogFunc);
 
 	mDriveThread.start((void *(*)(void*))driveLoop2, &gSipInterface );
 	mPeriodicServiceThread.start((void *(*)(void*))periodicServiceLoop, &gSipInterface );
diff --git a/SIP/SIPRtp.cpp b/SIP/SIPRtp.cpp
index ed8ed40c..12fb5f92 100644
--- a/SIP/SIPRtp.cpp
+++ b/SIP/SIPRtp.cpp
@@ -125,13 +125,13 @@ void SipRtp::initRTP1(const char *d_ip_addr, unsigned d_port, unsigned dialogId)
 	rtp_session_set_payload_type(mSession, 3);
 	// (pat added) The last argument is user payload data that is passed to ourRtpTimestampJumpCallback()
 	// I was going to use the dialogId but decided to look up the dialog by mSession.
-	rtp_session_signal_connect(mSession,"timestamp_jump",(RtpCallback)ourRtpTimestampJumpCallback,dialogId);
+	rtp_session_signal_connect(mSession,"timestamp_jump",(RtpCallback)ourRtpTimestampJumpCallback,(void *) dialogId);
 
 	gCountRtpSockets++;
 #ifdef ORTP_NEW_API
 	rtp_session_set_local_addr(mSession, "0.0.0.0", mRTPPort, -1);
 #else
-	rtp_session_set_local_addr(mSession, "0.0.0.0", mRTPPort);
+	rtp_session_set_local_addr(mSession, "0.0.0.0", mRTPPort, mRTPPort+1);
 #endif
 	rtp_session_set_remote_addr(mSession, d_ip_addr, d_port);
 	WATCHF("*** initRTP local=%d remote=%s %d\n",mRTPPort,d_ip_addr,d_port);
--- a/CommonLibs/Configuration.cpp
+++ b/CommonLibs/Configuration.cpp
@@ -30,6 +30,7 @@
 #include "Utils.h"
 #include <fstream>
 #include <iostream>
+#include <sys/types.h>
 #include <string.h>
 
 #ifdef DEBUG_CONFIG
--- a/CommonLibs/Logger.h
+++ b/CommonLibs/Logger.h
@@ -43,6 +43,7 @@
 #include <map>
 #include <string>
 #include <assert.h>
+#include <unistd.h>
 #include <sys/syscall.h>
 // We cannot include Utils.h because it includes Logger.h, so just declare timestr() here.
 // If timestr decl is changed G++ will whine when Utils.h is included.
