From 640cc3b6ca8003a867e98956a829f5134202d1fc Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 24 Oct 2025 14:57:11 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 24a2bba..d1fbfc3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=61", "hatch", "jupyter_packaging~=0.10.0", "wheel"]
+requires = ["setuptools>=61", "hatch", "jupyter_packaging", "wheel"]
 build-backend = "setuptools.build_meta"
 
 [project.urls]
