--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -530,7 +530,7 @@ if(TFLITE_ENABLE_XNNPACK)
   )
   target_link_libraries(xnnpack-delegate
     Eigen3::Eigen
-    flatbuffers::flatbuffers
+    FlatBuffers::FlatBuffers
     gemmlowp::gemmlowp
     ruy::ruy
     )
@@ -716,7 +716,7 @@ target_link_libraries(tensorflow-lite
     absl::variant
     farmhash
     fft2d_fftsg2d
-    flatbuffers::flatbuffers
+    FlatBuffers::FlatBuffers
     gemmlowp::gemmlowp
     ml_dtypes
     ruy::ruy
