From 168a26b4ded1e2a29c0e40b89c4a9e93c7675c6e Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 21 Feb 2025 14:19:53 +0800
Subject: [PATCH] Update xnnpack.cmake

---
 .../lite/tools/cmake/modules/xnnpack.cmake    | 27 -------------------
 1 file changed, 27 deletions(-)

diff --git a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
index 0bf6fc8..7af1d2e 100644
--- a/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
+++ b/tensorflow/lite/tools/cmake/modules/xnnpack.cmake
@@ -17,33 +17,6 @@ if(TARGET xnnpack OR xnnpack_POPULATED)
   return()
 endif()
 
-include(OverridableFetchContent)
-
-OverridableFetchContent_Declare(
-  xnnpack
-  GIT_REPOSITORY https://github.com/google/XNNPACK
-  # Sync with tensorflow/workspace2.bzl
-  GIT_TAG 6b83f69d4938da4dc9ad63c00bd13e9695659a51
-  GIT_PROGRESS TRUE
-  PREFIX "${CMAKE_BINARY_DIR}"
-  SOURCE_DIR "${CMAKE_BINARY_DIR}/xnnpack"
-)
-OverridableFetchContent_GetProperties(xnnpack)
-if(NOT xnnpack_POPULATED)
-  OverridableFetchContent_Populate(xnnpack)
-endif()
-
-# May consider setting XNNPACK_USE_SYSTEM_LIBS if we want to control all
-# dependencies by TFLite.
-set(XNNPACK_BUILD_TESTS OFF CACHE BOOL "Disable XNNPACK test.")
-set(XNNPACK_BUILD_BENCHMARKS OFF CACHE BOOL "Disable XNNPACK benchmarks.")
-
-# The following line adds project of PTHREADPOOL, FP16 and XNNPACK which are
-# needed to compile XNNPACK delegate of TFLite.
-# Note, we introduce an intermediate subdirectory, see ${TFLITE_SOURCE_DIR}/tools/cmake/modules/xnnpack/CMakeLists.txt
-# for details.
-add_subdirectory(${TFLITE_SOURCE_DIR}/tools/cmake/modules/xnnpack)
-
 include_directories(
   AFTER
    "${PTHREADPOOL_SOURCE_DIR}/include"
