From f66d3319862c1b62a7df6cb54573ac55bcd65808 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sat, 22 Aug 2026 18:57:28 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 tensorflow/lite/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index 7e877cd92f1cd1..b1773e57d4f153 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -51,7 +51,7 @@ if(NOT TENSORFLOW_SOURCE_DIR)
   FetchContent_Declare(
     tensorflow
     GIT_REPOSITORY https://github.com/tensorflow/tensorflow.git
-    GIT_TAG v2.19.0
+    GIT_TAG v2.21.0-rc0
     GIT_SHALLOW TRUE
     GIT_PROGRESS TRUE
     SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/tensorflow-src
@@ -886,4 +886,4 @@ if(TFLITE_ENABLE_XNNPACK)
     PUBLIC ${TFLITE_TARGET_PUBLIC_OPTIONS}
     PRIVATE ${TFLITE_TARGET_PRIVATE_OPTIONS}
   )
-endif()
\ No newline at end of file
+endif()

