From 3425c620b05b9b3dd683f6e61c68ce09f54391c2 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 25 Feb 2025 17:05:49 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 tensorflow/lite/CMakeLists.txt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index bce9627..499e0da 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -530,7 +530,6 @@ if(TFLITE_ENABLE_XNNPACK)
   )
   target_link_libraries(xnnpack-delegate
     Eigen3::Eigen
-    flatbuffers::flatbuffers
     gemmlowp::gemmlowp
     ruy::ruy
     )
@@ -716,7 +715,6 @@ target_link_libraries(tensorflow-lite
     absl::variant
     farmhash
     fft2d_fftsg2d
-    flatbuffers::flatbuffers
     gemmlowp::gemmlowp
     ml_dtypes
     ruy::ruy
@@ -810,6 +808,9 @@ set_target_properties(_pywrap_tensorflow_interpreter_wrapper PROPERTIES PREFIX "
 target_include_directories(_pywrap_tensorflow_interpreter_wrapper
   PUBLIC
     ${TENSORFLOW_SOURCE_DIR}
+    ${PYTHON_INCLUDE}
+    ${PYBIND11_INCLUDE}
+    ${NUMPY_INCLUDE}
 )
 
 target_link_libraries(_pywrap_tensorflow_interpreter_wrapper
