--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -710,6 +710,7 @@ target_link_libraries(tensorflow-lite
     Eigen3::Eigen
     absl::flags
     absl::hash
+    absl::log
     absl::status
     absl::strings
     absl::synchronization
--- a/tensorflow/lite/tools/cmake/modules/Findabsl.cmake
+++ b/tensorflow/lite/tools/cmake/modules/Findabsl.cmake
@@ -98,6 +98,8 @@ if(abseil-cpp_POPULATED)
     leak_check_api_disabled_for_testing
     leak_check_api_enabled_for_testing
     leak_check_disable
+    log_internal_message
+    log_internal_check_op
     log_severity
     main_lib
     malloc_internal
