From ef31493233127b810ee1a143ee561d69f437dea9 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 20 Feb 2025 12:28:15 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 tensorflow/lite/tools/benchmark/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorflow/lite/tools/benchmark/CMakeLists.txt b/tensorflow/lite/tools/benchmark/CMakeLists.txt
index 477254a..3279b6c 100644
--- a/tensorflow/lite/tools/benchmark/CMakeLists.txt
+++ b/tensorflow/lite/tools/benchmark/CMakeLists.txt
@@ -51,7 +51,6 @@ list(APPEND TFLITE_BENCHMARK_LIBS
   feature_proto
   example_proto
   model_runtime_info_proto
-  protobuf::libprotobuf
 )
 
 # TODO(b/171007016): Enable performance options on Windows.
