From d44770025a2b95bb1afd17faf0c41306f7eeffe4 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Mon, 9 Dec 2024 08:33:24 +0800
Subject: [PATCH] Add files via upload

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index bdcd0ce..d35cc42 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -62,5 +62,5 @@ python = ">=2.6 || >=3.0"
 pytest = "==6.0.1"
 
 [build-system]
-requires = ["poetry>=0.12"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=0.12"]
+build-backend = "poetry.core.masonry.api"
