From 9555348c9600da40300be536283f33c1dc80cf29 Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Wed, 26 Nov 2025 12:13:41 +0800
Subject: [PATCH] Update Cargo.toml

---
 data/Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Cargo.toml b/data/Cargo.toml
index 90baf801..6a877206 100644
--- a/data/Cargo.toml
+++ b/data/Cargo.toml
@@ -63,7 +63,7 @@ walkdir = { workspace = true }
 
 # Windows doesn't support assembly for compilation
 [target.'cfg(not(target_os = "windows"))'.dependencies]
-sha2 = { workspace = true, features = ["asm"] }
+sha2 = { workspace = true }
 
 [target.'cfg(target_os = "windows")'.dependencies]
 sha2 = { workspace = true }
