From 0e1630769bbc8d56d5616c9d82c192cb462ef3a6 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 14 Dec 2024 19:31:39 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index ef8ce79..8442e05 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools==75.1.0"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
