--- a/homeassistant/components/python_script/__init__.py
+++ b/homeassistant/components/python_script/__init__.py
@@ -132,7 +132,7 @@ def discover_scripts(hass):
     else:
         services_dict = {}
 
-    for fil in glob.iglob(os.path.join(path, "*.py")):
+    for fil in glob.iglob(os.path.join(path, "*.pyc")):
         name = os.path.splitext(os.path.basename(fil))[0]
         hass.services.register(
             DOMAIN,
@@ -182,7 +182,7 @@ def guarded_inplacevar(op: str, target:
 @bind_hass
 def execute_script(hass, name, data=None, return_response=False):
     """Execute a script."""
-    filename = f"{name}.py"
+    filename = f"{name}.pyc"
     raise_if_invalid_filename(filename)
     with open(hass.config.path(FOLDER, filename), encoding="utf8") as fil:
         source = fil.read()
--- a/homeassistant/loader.py
+++ b/homeassistant/loader.py
@@ -1239,7 +1239,7 @@ def platforms_exists(self, platform_names: Iterable[str]) -> list[str]:
         for platform_name in platform_names:
             full_name = f"{domain}.{platform_name}"
             if full_name not in missing_platforms and (
-                f"{platform_name}.py" in files or platform_name in files
+                f"{platform_name}.pyc" in files or platform_name in files
             ):
                 existing_platforms.append(platform_name)
                 continue
--- a/homeassistant/scripts/__init__.py
+++ b/homeassistant/scripts/__init__.py
@@ -29,7 +29,7 @@ def run(args: list[str]) -> int:
 
         if os.path.isdir(os.path.join(path, fil)):
             scripts.append(fil)
-        elif fil != "__init__.py" and fil.endswith(".py"):
+        elif fil != "__init__.pyc" and fil.endswith(".pyc"):
             scripts.append(fil[:-3])
 
     if not args:
 
From e8b44e8de34c1b5ad76d8180a258de0270a7825a Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 13 Nov 2025 20:46:38 +0800
Subject: [PATCH] Update setup.py

---
 homeassistant/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/homeassistant/setup.py b/homeassistant/setup.py
index a631eb07ca2820..1a6f510c3f7ffe 100644
--- a/homeassistant/setup.py
+++ b/homeassistant/setup.py
@@ -666,9 +666,9 @@ class SetupPhases(StrEnum):
     """Constants for setup time measurements."""
 
     SETUP = "setup"
-    """Set up of a component in __init__.py."""
+    """Set up of a component in __init__.pyc."""
     CONFIG_ENTRY_SETUP = "config_entry_setup"
-    """Set up of a config entry in __init__.py."""
+    """Set up of a config entry in __init__.pyc."""
     PLATFORM_SETUP = "platform_setup"
     """Set up of a platform integration.
 
