--- a/homeassistant/components/python_script/__init__.py
+++ b/homeassistant/components/python_script/__init__.py
@@ -132,7 +132,7 @@ def discover_scripts(hass):
     else:
         services_dict = {}
 
-    for fil in glob.iglob(os.path.join(path, "*.py")):
+    for fil in glob.iglob(os.path.join(path, "*.pyc")):
         name = os.path.splitext(os.path.basename(fil))[0]
         hass.services.register(
             DOMAIN,
@@ -182,7 +182,7 @@ def guarded_inplacevar(op: str, target:
 @bind_hass
 def execute_script(hass, name, data=None, return_response=False):
     """Execute a script."""
-    filename = f"{name}.py"
+    filename = f"{name}.pyc"
     raise_if_invalid_filename(filename)
     with open(hass.config.path(FOLDER, filename), encoding="utf8") as fil:
         source = fil.read()
--- a/homeassistant/loader.py
+++ b/homeassistant/loader.py
@@ -1239,7 +1239,7 @@ def platforms_exists(self, platform_names: Iterable[str]) -> list[str]:
         for platform_name in platform_names:
             full_name = f"{domain}.{platform_name}"
             if full_name not in missing_platforms and (
-                f"{platform_name}.py" in files or platform_name in files
+                f"{platform_name}.pyc" in files or platform_name in files
             ):
                 existing_platforms.append(platform_name)
                 continue
--- a/homeassistant/scripts/__init__.py
+++ b/homeassistant/scripts/__init__.py
@@ -28,6 +28,8 @@ def run(args: list[str]) -> int:
 
         if os.path.isdir(os.path.join(path, fil)):
             scripts.append(fil)
+        elif fil != "__init__.pyc" and fil.endswith(".pyc"):
+            scripts.append(fil[:-4])
         elif fil != "__init__.py" and fil.endswith(".py"):
             scripts.append(fil[:-3])
 
 
From e8b44e8de34c1b5ad76d8180a258de0270a7825a Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 13 Nov 2025 20:46:38 +0800
Subject: [PATCH] Update setup.py

---
 homeassistant/setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/homeassistant/setup.py b/homeassistant/setup.py
index a631eb07ca2820..1a6f510c3f7ffe 100644
--- a/homeassistant/setup.py
+++ b/homeassistant/setup.py
@@ -666,9 +666,9 @@ class SetupPhases(StrEnum):
     """Constants for setup time measurements."""
 
     SETUP = "setup"
-    """Set up of a component in __init__.py."""
+    """Set up of a component in __init__.pyc."""
     CONFIG_ENTRY_SETUP = "config_entry_setup"
-    """Set up of a config entry in __init__.py."""
+    """Set up of a config entry in __init__.pyc."""
     PLATFORM_SETUP = "platform_setup"
     """Set up of a platform integration.
 
From 69d16f540a217d9be1ca7ff778449ce351b43d20 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 13 Nov 2025 21:14:20 +0800
Subject: [PATCH] Update block_async_io.py

---
 homeassistant/block_async_io.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/homeassistant/block_async_io.py b/homeassistant/block_async_io.py
index eb81268434bf7..a7b5f06c2f9ff 100644
--- a/homeassistant/block_async_io.py
+++ b/homeassistant/block_async_io.py
@@ -46,7 +46,7 @@ def _check_sleep_call_allowed(mapped_args: dict[str, Any]) -> bool:
     # frame[2] is protected_loop_func
     # frame[3] is the offender
     with suppress(ValueError):
-        return get_current_frame(4).f_code.co_filename.endswith("pydevd.py")
+        return get_current_frame(4).f_code.co_filename.endswith("pydevd.pyc")
     return False
 
 
From 06cfc66f069c9264440fc906ee944adaa0599a8a Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 13 Nov 2025 21:49:58 +0800
Subject: [PATCH] Update system_info.py

---
 homeassistant/helpers/system_info.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/homeassistant/helpers/system_info.py b/homeassistant/helpers/system_info.py
index 1baec4df052a68..2ff21e2170c43a 100644
--- a/homeassistant/helpers/system_info.py
+++ b/homeassistant/helpers/system_info.py
@@ -66,7 +66,7 @@ async def async_get_system_info(hass: HomeAssistant) -> dict[str, Any]:
     is_hassio_ = is_hassio(hass)
 
     info_object = {
-        "installation_type": "Unknown",
+        "installation_type": "OpenWrt",
         "version": current_version,
         "dev": "dev" in current_version,
         "hassio": is_hassio_,
