From 381a7dcb92ca7f242768dfb7f34bbdeda09d53aa Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 1 Mar 2026 07:09:26 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 240b47f..5b77fc2 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
-    "setuptools>=68.0,<82.1",
-    "wheel>=0.40,<0.47",
+    "setuptools",
+    "wheel",
 ]
 build-backend = "setuptools.build_meta"
 
