From 54ef1b3878fdcf2ebc1449b9932391853b30576f Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Mon, 26 Jan 2026 19:42:53 +0800
Subject: [PATCH] Update client.py

---
 onvif/client.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/onvif/client.py b/onvif/client.py
index 55df04c..cf18ace 100644
--- a/onvif/client.py
+++ b/onvif/client.py
@@ -45,7 +45,7 @@
 logging.getLogger("zeep.client").setLevel(logging.CRITICAL)
 
 _SENTINEL = object()
-_WSDL_PATH = os.path.join(os.path.dirname(os.path.dirname(__file__)), "wsdl")
+_WSDL_PATH = /usr/lib/python3.14/site-packages/onvif/wsdl
 _DEFAULT_TIMEOUT = 90
 _PULLPOINT_TIMEOUT = 90
 _CONNECT_TIMEOUT = 30
