From e94afc467e82a5a07c988b55dc2209d3c63d5480 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sun, 8 Feb 2026 14:23:10 +0800
Subject: [PATCH] Update Makefile.PL

---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index f9c622e..270e4e9 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -156,7 +156,7 @@ push(@libs, '-lcrypt') if ($Config{osname} =~ /freebsd/i);
 #push(@lddflags, '-static') if ($Config{osname} =~ /cygwin/i);
 
 # Check whether xmalloc has been renamed to _rl_malloc (Gentoo Linux)
-if (my $extra_defs = guess_malloc_names("@RLINC", "@RLLIB", "@defs", "@lddflags", "@libs")) {
+if(my $extra_defs=""){
     push(@defs, $extra_defs);
 }
 
