From d08316e57179e3594ec332ae1767a9b042d48ec2 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sun, 22 Mar 2026 17:24:33 +0800
Subject: [PATCH] Update include directories to use STAGING_DIR

---
 lib/ModPerl/CScan.pm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/ModPerl/CScan.pm b/lib/ModPerl/CScan.pm
index 4985ef09d..2a18fcf18 100644
--- a/lib/ModPerl/CScan.pm
+++ b/lib/ModPerl/CScan.pm
@@ -80,7 +80,7 @@ my $recipes
       includeDirs =>  { prerequisites => ['filedir'],
                         output => sub {
                           my $data = shift;
-                          [ $data->{filedir}, '/usr/local/include', '.'];
+                          [ $data->{filedir}, '@STAGING_DIR@', '.'];
                         } },
       Cpp => { prerequisites => [qw(cppminus add_cppflags cppflags cppstdin)],
                output => sub {
@@ -939,7 +939,7 @@ sub new {
       = (shift, shift, shift, shift, shift);
     $Cpp ||= \%Config::Config;
     my $filedir = dirname $filename || '.';
-    $Includes ||= [$filedir, '/usr/local/include', '.'];
+    $Includes ||= [$filedir, '@STAGING_DIR@', '.'];
     my $addincludes = "";
     $addincludes = "-I" . join(" -I", @$Includes)
       if defined $Includes and @$Includes;
