From 7e867ac0b9ebed7255e290cc91459927ad0f79f4 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 16 Oct 2025 07:51:04 +0800
Subject: [PATCH] Add files via upload

---
 configure    | 2 +-
 configure.ac | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index d31c24d..7b4daf1 100644
--- a/configure
+++ b/configure
@@ -32568,7 +32568,7 @@ fi
 # LIBHACL_CFLAGS: compiler flags passed for HACL* and HACL*-based modules
 # LIBHACL_LDFLAGS: linker flags passed for HACL* and HACL*-based modules
 LIBHACL_FLAG_I='-I$(srcdir)/Modules/_hacl -I$(srcdir)/Modules/_hacl/include'
-LIBHACL_FLAG_D='-D_BSD_SOURCE -D_DEFAULT_SOURCE'
+LIBHACL_FLAG_D=''
 case "$ac_sys_system" in
   Linux*)
     if test "$ac_cv_func_explicit_bzero" = "no"; then
diff --git a/configure.ac b/configure.ac
index af7a962..3ea78c3 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7995,7 +7995,7 @@ PY_STDLIB_MOD_SIMPLE([unicodedata])
 # LIBHACL_CFLAGS: compiler flags passed for HACL* and HACL*-based modules
 # LIBHACL_LDFLAGS: linker flags passed for HACL* and HACL*-based modules
 LIBHACL_FLAG_I='-I$(srcdir)/Modules/_hacl -I$(srcdir)/Modules/_hacl/include'
-LIBHACL_FLAG_D='-D_BSD_SOURCE -D_DEFAULT_SOURCE'
+LIBHACL_FLAG_D=''
 case "$ac_sys_system" in
   Linux*)
     if test "$ac_cv_func_explicit_bzero" = "no"; then

From 97bccbbe354bf6a3232b963241bcdc6a9bedc78f Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 16 Oct 2025 08:31:41 +0800
Subject: [PATCH] Update lowstar_endianness.h

---
 Modules/_hacl/include/krml/lowstar_endianness.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/_hacl/include/krml/lowstar_endianness.h b/Modules/_hacl/include/krml/lowstar_endianness.h
index af6b882..9e5d5cf 100644
--- a/Modules/_hacl/include/krml/lowstar_endianness.h
+++ b/Modules/_hacl/include/krml/lowstar_endianness.h
@@ -6,6 +6,7 @@
 
 #include <string.h>
 #include <inttypes.h>
+#include <endian.h>
 
 /******************************************************************************/
 /* Implementing C.fst (part 2: endian-ness macros)                            */
