From a4948b0733940943990b493dde28c361a59cb92d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 16 Oct 2025 12:06:57 +0800
Subject: [PATCH] Update _ssl.c

---
 Modules/_ssl.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Modules/_ssl.c b/Modules/_ssl.c
index 21c3991..25d3079 100644
--- a/Modules/_ssl.c
+++ b/Modules/_ssl.c
@@ -62,6 +62,7 @@
 #include "openssl/rsa.h"
 #include "openssl/crypto.h"
 #include "openssl/x509.h"
+#include "openssl/x509_vfy.h"
 #include "openssl/x509v3.h"
 #include "openssl/pem.h"
 #include "openssl/ssl.h"
