From b199b5a0c59ef67543c570172d40c5b46795e28e Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Mon, 27 Jan 2025 19:20:49 +0800
Subject: [PATCH] Update sha2.c

Signed-off-by: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
---
 sha2/sha2.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sha2/sha2.c b/sha2/sha2.c
index 9acec04c..0f4cbcdf 100644
--- a/sha2/sha2.c
+++ b/sha2/sha2.c
@@ -43,6 +43,7 @@
 #endif
 #endif
 #include "sha2.h"
+#include <stdint.h>
 
 /*
  * ASSERT NOTE:
