From 3fbb13496f2c885066a7d72794307a580640e493 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 25 Dec 2025 14:41:11 +0800
Subject: [PATCH] Update encoder.h

---
 src/encoder.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/encoder.h b/src/encoder.h
index 5b85e5f..a794655 100644
--- a/src/encoder.h
+++ b/src/encoder.h
@@ -42,7 +42,9 @@
  * Types
  ****************************************************************************/
 
-typedef int bool;
+#if __STDC_VERSION__ < 202311L
+ typedef int bool;
+#endif
 
 /*****************************************************************************
  * Structures
