From 97a5bcf0c905d978135edf6e7c6848039e092089 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Mon, 3 Feb 2025 15:01:23 +0800
Subject: [PATCH] Update setup_configure.py

---
 setup_configure.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup_configure.py b/setup_configure.py
index 3c5fe0d..046a8dd 100644
--- a/setup_configure.py
+++ b/setup_configure.py
@@ -286,10 +286,8 @@ def _load_hdf5_lib(self, libdirs):
             raise FileNotFoundError(f"{path} is missing")
 
         try:
-            lib = ctypes.CDLL(path, **load_kw)
+            lib = ctypes.CDLL(path)
         except Exception:
-            print("error: Unable to load dependency HDF5, make sure HDF5 is installed properly")
-            print(f"on {sys.platform=} with {platform.machine()=}")
             print("Library dirs checked:", libdirs)
             raise
 
