From 972fe460ca3ce73e53946bb69d5bb86e57efdaf9 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 2 Feb 2025 18:25:01 +0800
Subject: [PATCH] Update setup_build.py

---
 setup_build.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup_build.py b/setup_build.py
index c0e9a28..d226db6 100644
--- a/setup_build.py
+++ b/setup_build.py
@@ -56,8 +56,8 @@ def localpath(*args):
         ('H5_BUILT_AS_DYNAMIC_LIB', None)
     ])
 else:
-    FALLBACK_PATHS['include_dirs'].extend(['/opt/local/include', '/usr/local/include'])
-    FALLBACK_PATHS['library_dirs'].extend(['/opt/local/lib', '/usr/local/lib'])
+    FALLBACK_PATHS['include_dirs'].extend(['@STAGING_DIR@/usr/include'])
+    FALLBACK_PATHS['library_dirs'].extend(['@STAGING_DIR@/usr/lib'])
 
 
 class h5py_build_ext(build_ext):
