From 23cc17d921e45777587a8266b0772ccd8493ee1c Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 7 Jan 2025 17:06:03 +0800
Subject: [PATCH] Update setup_configure.py

---
 setup_configure.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup_configure.py b/setup_configure.py
index 3c5fe0d..e2ccb0d 100644
--- a/setup_configure.py
+++ b/setup_configure.py
@@ -144,8 +144,8 @@ def _find_hdf5_compiler_settings(mpi=False):
 
         # Specified a prefix dir (e.g. '/usr/local')
         if hdf5:
-            inc_dirs = [op.join(hdf5, 'include')]
-            lib_dirs = [op.join(hdf5, 'lib')]
+            inc_dirs = [op.join(hdf5, '@STAGING_DIR@')]
+            lib_dirs = [op.join(hdf5, '@STAGING_DIR@')]
             if sys.platform.startswith('win'):
                 lib_dirs.append(op.join(hdf5, 'bin'))
             return (inc_dirs, lib_dirs, [])
