From 79399abdb22305ec8ecf67780e5a8931bf01223d Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Sat, 29 Nov 2025 06:51:50 +0800
Subject: [PATCH] Update _backend.py

---
 packaging/pep517_backend/_backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/pep517_backend/_backend.py b/packaging/pep517_backend/_backend.py
index a163f9c..32dda42 100644
--- a/packaging/pep517_backend/_backend.py
+++ b/packaging/pep517_backend/_backend.py
@@ -376,7 +376,7 @@ def get_requires_for_build_wheel(
     if is_pure_python_build:
         c_ext_build_deps = []
     else:
-        c_ext_build_deps = ['Cython ~= 3.1.0']
+        c_ext_build_deps = ['Cython']
 
     return _setuptools_get_requires_for_build_wheel(
         config_settings=config_settings,
