From 15442a38dd9111fa294f76d05617b41c68d75d73 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Wed, 28 Jan 2026 16:00:23 +0800
Subject: [PATCH] Update interpreter.py

---
 mesonbuild/interpreter/interpreter.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mesonbuild/interpreter/interpreter.py b/mesonbuild/interpreter/interpreter.py
index 09f828726ef6..23d4db38ac5c 100644
--- a/mesonbuild/interpreter/interpreter.py
+++ b/mesonbuild/interpreter/interpreter.py
@@ -887,8 +887,6 @@ def do_subproject(self, subp_name: str, kwargs: kwtypes.DoSubproject, force_meth
             default_options = [default_options]
         if isinstance(default_options, list):
             default_options = dict((x.split('=', 1) for x in default_options))
-        if extra_default_options:
-            default_options = {**extra_default_options, **default_options}
 
         # This in practice is only used for default_library.  forced_options is the
         # only case in which a meson.build file overrides the machine file or the
