From 28dc9e11f01a26e65a2c47a2fc65258483e4211d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 26 Jun 2026 21:27:57 +0800
Subject: [PATCH] Update GNUmakefile

---
 GNUmakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GNUmakefile b/GNUmakefile
index da282921..66b25707 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -22,7 +22,7 @@ endif
 # `make SNOWBALL_FLAGS=-comments` to generate target language code with
 # comments indicating the corresponding lines in the .sbl source.
 SNOWBALL_FLAGS ?=
-SNOWBALL_COMPILE := ./snowball $(SNOWBALL_FLAGS)
+SNOWBALL_COMPILE := snowball $(SNOWBALL_FLAGS)
 
 # Ada
 
