From 0487e3e582012846be6c17f8deabda7890bbac10 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Thu, 28 Aug 2025 15:24:10 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 td/generate/CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/td/generate/CMakeLists.txt b/td/generate/CMakeLists.txt
index 9e71ad2a1101..c41c3a1a9ea5 100644
--- a/td/generate/CMakeLists.txt
+++ b/td/generate/CMakeLists.txt
@@ -185,6 +185,14 @@ if (NOT CMAKE_CROSSCOMPILING)
   endif()
 
   add_executable(generate_mtproto ${TL_GENERATE_MTPROTO_SOURCE})
+
+if(DEFINED TL_GENERATE_MTPROTO_CXX_COMPILER)
+	set(CMAKE_CXX_COMPILER "${TL_GENERATE_MTPROTO_CXX_COMPILER}")
+	set(CMAKE_CXX_FLAGS "")
+	set(CMAKE_EXE_LINKER_FLAGS "")
+	set(CMAKE_SYSROOT "")
+endif()
+
   target_link_libraries(generate_mtproto PRIVATE tdtl tl_writer_cpp)
   add_custom_target(tl_generate_mtproto
     WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/auto
