From 46b9885d66976abf47052529824dd74f2420601c Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 17 Sep 2025 01:28:13 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 python/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/CMakeLists.txt b/python/CMakeLists.txt
index 748921a5..01cc259d 100644
--- a/python/CMakeLists.txt
+++ b/python/CMakeLists.txt
@@ -37,7 +37,7 @@ add_custom_target(
     ALL
     DEPENDS tbb python_copy
     COMMAND
-    ${PYTHON_EXECUTABLE} ${PYTHON_BUILD_WORK_DIR}/setup.py
+    ${PYTHON3_VARS} ${PYTHON_EXECUTABLE} ${PYTHON_BUILD_WORK_DIR}/setup.py
         build -b${PYTHON_BUILD_WORK_DIR}
         build_ext ${TBB4PY_INCLUDE_STRING} -L$<TARGET_FILE_DIR:TBB::tbb>
         install --prefix build -f
