From 0274ab1f7393537a37aa37dbe95ae83ce975ee23 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 10 Dec 2025 06:20:49 +0800
Subject: [PATCH] Update ucp_types.h

---
 src/ucp/core/ucp_types.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/ucp/core/ucp_types.h b/src/ucp/core/ucp_types.h
index 67b2ee87fd3..8476facd39c 100644
--- a/src/ucp/core/ucp_types.h
+++ b/src/ucp/core/ucp_types.h
@@ -8,7 +8,6 @@
 #define UCP_TYPES_H_
 
 #include <ucp/api/ucp.h>
-#include <ucs/type/float8.h>
 #include <uct/api/uct.h>
 #include <ucs/datastruct/static_bitmap.h>
 #include <ucs/sys/preprocessor.h>
