From cc116c9a65b156e7c31add5690bb1c7da34bc80a Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 10 Dec 2025 06:28:09 +0800
Subject: [PATCH] Update float8.h

---
 src/ucs/type/float8.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/ucs/type/float8.h b/src/ucs/type/float8.h
index 84cd819689d..fca889f76e8 100644
--- a/src/ucs/type/float8.h
+++ b/src/ucs/type/float8.h
@@ -7,8 +7,6 @@
 #ifndef UCS_TYPE_FLOAT_H
 #define UCS_TYPE_FLOAT_H
 
-#include <ieee754.h>
-
 #include <ucs/sys/preprocessor.h>
 #include <ucs/debug/assert.h>
 #include <ucs/arch/bitops.h>
