From 6d074a9d13b7113a6fd560072259786afa40fec9 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Tue, 9 Dec 2025 23:18:34 +0800
Subject: [PATCH] Update numa.c

---
 src/ucs/memory/numa.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/ucs/memory/numa.c b/src/ucs/memory/numa.c
index 57d29f17db9..d9f077aab3f 100644
--- a/src/ucs/memory/numa.c
+++ b/src/ucs/memory/numa.c
@@ -114,7 +114,7 @@ unsigned ucs_numa_num_configured_cpus()
 
     if (num_cpus == 0) {
         max_cpu  = ucs_numa_get_max_dirent(UCS_SYS_FS_CPUS_PATH, "cpu",
-                                           __CPU_SETSIZE, 0);
+                                           CPU_SETSIZE, 0);
         num_cpus = max_cpu + 1;
     }
 
@@ -124,12 +124,12 @@ unsigned ucs_numa_num_configured_cpus()
 ucs_numa_node_t ucs_numa_node_of_cpu(int cpu)
 {
     /* Used for caching to improve performance */
-    static ucs_numa_node_t cpu_numa_node[__CPU_SETSIZE] = {0};
+    static ucs_numa_node_t cpu_numa_node[CPU_SETSIZE] = {0};
     char *core_dir_path;
     ucs_numa_node_t node;
     ucs_status_t status;
 
-    ucs_assert(cpu < __CPU_SETSIZE);
+    ucs_assert(cpu < CPU_SETSIZE);
 
     if (cpu_numa_node[cpu] == 0) {
         status = ucs_string_alloc_formatted_path(&core_dir_path,
--- a/src/ucs/config/global_opts.c
+++ b/src/ucs/config/global_opts.c
@@ -18,7 +18,7 @@
 #include <ucs/sys/string.h>
 #include <ucs/vfs/base/vfs_obj.h>
 #include <ucs/vfs/sock/vfs_sock.h>
-#include <sys/signal.h>
+#include <signal.h>
 
 
 ucs_global_opts_t ucs_global_opts = {
