From 2059a1fe1d21a598ec6f906e403786d845a3ea0f Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Tue, 9 Dec 2025 12:10:50 +0800
Subject: [PATCH] Update reloc.h

---
 src/ucm/util/reloc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ucm/util/reloc.h b/src/ucm/util/reloc.h
index f684ea26ba6..e2cbb21e0ef 100644
--- a/src/ucm/util/reloc.h
+++ b/src/ucm/util/reloc.h
@@ -11,6 +11,7 @@
 #include <ucs/datastruct/list.h>
 #include <ucs/type/status.h>
 #include <ucm/util/log.h>
+#define _GNU_SOURCE
 #include <dlfcn.h>
 
 
