From edb9a09cc836ba77eec782022d08da1cb77325da Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 9 Dec 2025 01:59:31 +0800
Subject: [PATCH] Update reloc.c

---
 src/ucm/util/reloc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/ucm/util/reloc.c b/src/ucm/util/reloc.c
index bcf246fdce7..79bc55b27f7 100644
--- a/src/ucm/util/reloc.c
+++ b/src/ucm/util/reloc.c
@@ -8,6 +8,9 @@
 #  include "config.h"
 #endif
 
+#define _GNU_SOURCE
+#include <dlfcn.h> // The header file that defines RTLD_DI_SERINFO
+
 #include "reloc.h"
 
 #include <ucm/util/khash_safe.h>
