From 4b3fd1d46855513fa0c7df37936b5c449289ee05 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 28 Apr 2026 10:34:40 +0800
Subject: [PATCH] Update override.hpp

---
 boost/python/override.hpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/boost/python/override.hpp b/boost/python/override.hpp
index 3971425..b631226 100644
--- a/boost/python/override.hpp
+++ b/boost/python/override.hpp
@@ -97,7 +97,7 @@ class override : public object
     operator()() const
     {
         detail::method_result x(
-            PyEval_CallFunction(
+            PyObject_CallFunction(
                 this->ptr()
               , const_cast<char*>("()")
             ));
@@ -132,7 +132,7 @@ detail::method_result
 operator()( BOOST_PP_ENUM_BINARY_PARAMS_Z(1, N, A, const& a) ) const
 {
     detail::method_result x(
-        PyEval_CallFunction(
+        PyObject_CallFunction(
             this->ptr()
           , const_cast<char*>("(" BOOST_PP_REPEAT_1ST(N, BOOST_PYTHON_FIXED, "O") ")")
             BOOST_PP_REPEAT_1ST(N, BOOST_PYTHON_fast_arg_to_python_get, nil)
--- a/boost/python/call.hpp
+++ b/boost/python/call.hpp
@@ -60,7 +60,7 @@ call(PyObject* callable
     )
 {
     PyObject* const result = 
-        PyEval_CallFunction(
+        PyObject_CallFunction(
             callable
             , const_cast<char*>("(" BOOST_PP_REPEAT_1ST(N, BOOST_PYTHON_FIXED, "O") ")")
             BOOST_PP_REPEAT_1ST(N, BOOST_PYTHON_FAST_ARG_TO_PYTHON_GET, nil)
@@ -69,7 +69,7 @@ call(PyObject* callable
     // This conversion *must not* be done in the same expression as
     // the call, because, in the special case where the result is a
     // reference a Python object which was created by converting a C++
-    // argument for passing to PyEval_CallFunction, its reference
+    // argument for passing to PyObject_CallFunction, its reference
     // count will be 2 until the end of the full expression containing
     // the conversion, and that interferes with dangling
     // pointer/reference detection.

