From 0d032a4490895dffeba72460981c20b3ef5ff9bd Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 10 Jan 2025 10:46:50 +0800
Subject: [PATCH] Update setup.py

---
 setup.py | 30 +++++++++++++++++++++++++++++-
 1 file changed, 29 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 48d5a65..087f089 100644
--- a/setup.py
+++ b/setup.py
@@ -195,7 +195,35 @@ def main():
             "-DBUILD_PERF_TESTS=OFF",
             "-DBUILD_DOCS=OFF",
             "-DPYTHON3_LIMITED_API=ON",
-            "-DBUILD_OPENEXR=ON",
+            "-DBUILD_ZLIB=OFF",
+            "-DBUILD_TIFF=OFF",
+            "-DBUILD_OPENJPEG=OFF",
+            "-DBUILD_JASPER=OFF",
+            "-DBUILD_JPEG=OFF",
+            "-DBUILD_PNG=OFF",
+            "-DBUILD_OPENEXR=OFF",
+            "-DBUILD_WEBP=OFF",
+            "-DBUILD_TBB=OFF",
+            "-DBUILD_IPP_IW=OFF",
+            "-DBUILD_ITT=OFF",
+            "-DBUILD_JAVA=OFF",
+            "-DWITH_TIFF=ON",
+            "-DWITH_OPENJPEG=ON",
+            "-DWITH_JASPER=OFF",
+            "-DWITH_JPEG=ON",
+            "-DWITH_PNG=ON",
+            "-DWITH_OPENEXR=OFF",
+            "-DWITH_WEBP=ON",
+            "-DWITH_TBB=OFF",
+            "-DWITH_IPP=OFF",
+            "-DWITH_ITT=OFF",
+            "-DWITH_GTK=OFF",
+            "-DWITH_ADE=OFF",
+            "-DWITH_PROTOBUF=ON",
+            "-DWITH_VA=OFF",
+            "-DWITH_VA_INTEL=OFF",
+            "-DWITH_VTK=OFF",
+            "-DWITH_GPHOTO2=ON",
         ]
         + (
             # CMake flags for windows/arm64 build
