--- a/apps/sipauthserve.cpp
+++ b/apps/sipauthserve.cpp
@@ -32,8 +32,6 @@
 #include <fstream>
 #include <iostream>
 #include <netinet/in.h>
-#include <osip2/osip.h>
-#include <osipparser2/osip_message.h>
 #include <stdlib.h>
 #include <string.h>
 #include <sys/socket.h>
@@ -42,6 +40,8 @@
 #include <sys/types.h>
 #include <time.h>
 #include <unistd.h>
+#include <osip2/osip.h>
+#include <osipparser2/osip_message.h>
 
 #include <Logger.h>
 #include <Globals.h>
--- a/CommonLibs/Configuration.cpp
+++ b/CommonLibs/Configuration.cpp
@@ -30,6 +30,7 @@
 #include "Utils.h"
 #include <fstream>
 #include <iostream>
+#include <sys/types.h>
 #include <string.h>
 
 #ifdef DEBUG_CONFIG
--- a/CommonLibs/Logger.h
+++ b/CommonLibs/Logger.h
@@ -43,6 +43,7 @@
 #include <map>
 #include <string>
 #include <assert.h>
+#include <unistd.h>
 #include <sys/syscall.h>
 // We cannot include Utils.h because it includes Logger.h, so just declare timestr() here.
 // If timestr decl is changed G++ will whine when Utils.h is included.
