From 40dbdb287fa02f13edc61bf7ee76b963ab61b037 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 2 Apr 2026 05:58:57 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 170519154..7c1637240 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -71,5 +71,5 @@ custom-hook = "pdm_build.py"
 moonraker = "moonraker.server:main"
 
 [build-system]
-requires = ["pdm-backend<=2.4.4"]
+requires = ["pdm-backend"]
 build-backend = "pdm.backend"
