From 0c4bbc21e64b425e36b4ee9ac2a538529dda6e8e Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 24 Oct 2025 19:37:43 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index be6ef65..f3c74a7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools~=77.0"]
+requires = ["setuptools"]
 build-backend = "setuptools.build_meta"
 
 [project]
