--- a/lib/src/mac/mac_sch_pdu_nr.cc
+++ b/lib/src/mac/mac_sch_pdu_nr.cc
@@ -20,6 +20,7 @@
  */
 
 #include "srsran/mac/mac_sch_pdu_nr.h"
+#include <endian.h>
 
 namespace srsran {
 
--- a/lib/src/mac/pdu.cc
+++ b/lib/src/mac/pdu.cc
@@ -23,6 +23,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
+#include <endian.h>
 
 #include "srsran/common/standard_streams.h"
 #include "srsran/mac/pdu.h"
--- a/lib/src/common/threads.c
+++ b/lib/src/common/threads.c
@@ -21,6 +21,7 @@
 
 #include <errno.h>
 #include <pthread.h>
+#include <sched.h> // Needed for cpu_set_t and CPU_SET macros
 #include <stdbool.h>
 #include <stdio.h>
 #include <string.h>
@@ -141,10 +142,11 @@ bool threads_new_rt_cpu(pthread_t* thread, void* (*start_routine)(void*), void*
       CPU_ZERO(&cpuset);
       CPU_SET((size_t)cpu, &cpuset);
     }
-
+#ifdef __GLIBC__
     if (pthread_attr_setaffinity_np(&attr, sizeof(cpu_set_t), &cpuset)) {
       perror("pthread_attr_setaffinity_np");
     }
+#endif
   }
 
 // TSAN seems to have issues with thread attributes when running as normal user, disable them in that case
--- a/lib/include/srsran/common/string_helpers.h
+++ b/lib/include/srsran/common/string_helpers.h
@@ -29,6 +29,8 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <sys/types.h>
+#include <stdint.h>
 
 namespace srsran {
 
--- a/lib/include/srsran/common/backward.hpp
+++ b/lib/include/srsran/common/backward.hpp
@@ -207,8 +207,9 @@
 #endif
 #include <signal.h>
 #include <sys/stat.h>
-#include <syscall.h>
+#include <sys/syscall.h>
 #include <unistd.h>
+#include <dlfcn.h>
 
 #if BACKWARD_HAS_BFD == 1
 //              NOTE: defining PACKAGE{,_VERSION} is required before including
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -263,7 +263,7 @@ set(BOOST_REQUIRED_COMPONENTS
   program_options
 )
 if(UHD_FOUND) # Ubuntu 18.04 requires component 'system' to link UHD
-  list(APPEND BOOST_REQUIRED_COMPONENTS "system")
+  list(APPEND BOOST_REQUIRED_COMPONENTS)
 endif(UHD_FOUND)
 if(UNIX AND EXISTS "/usr/lib64")
   list(APPEND BOOST_LIBRARYDIR "/usr/lib64") #fedora 64-bit fix
--- a/lib/src/srslog/sinks/stream_sink.h
+++ b/lib/src/srslog/sinks/stream_sink.h
@@ -22,6 +22,16 @@
 #ifndef SRSLOG_STREAM_SINK_H
 #define SRSLOG_STREAM_SINK_H
 
+#include <stdio.h> // Or <stdio.h>
+
+#ifdef stdout
+#undef stdout
+#endif
+
+#ifdef stderr
+#undef stderr
+#endif
+
 #include "srsran/srslog/sink.h"
 
 namespace srslog {
--- a/lib/src/phy/io/netsource.c
+++ b/lib/src/phy/io/netsource.c
@@ -27,6 +27,7 @@
 #include <stdlib.h>
 #include <strings.h>
 #include <sys/socket.h>
+#include <sys/time.h>
 #include <unistd.h>
 
 #include "srsran/phy/io/netsource.h"
--- a/lib/src/phy/io/netsink.c
+++ b/lib/src/phy/io/netsink.c
@@ -84,7 +84,7 @@ int srsran_netsink_set_nonblocking(srsran_netsink_t* q)
 int srsran_netsink_write(srsran_netsink_t* q, void* buffer, int nof_bytes)
 {
   if (!q->connected) {
-    if (connect(q->sockfd, &q->servaddr, sizeof(q->servaddr)) < 0) {
+    if (connect(q->sockfd, (struct sockaddr *)&q->servaddr, sizeof(q->servaddr)) < 0) {
       if (errno == ECONNREFUSED || errno == EINPROGRESS) {
         return SRSRAN_SUCCESS;
       } else {
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ endif(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
 project( SRSRAN )
 message( STATUS "CMAKE_SYSTEM: " ${CMAKE_SYSTEM} )
 message( STATUS "CMAKE_SYSTEM_PROCESSOR: " ${CMAKE_SYSTEM_PROCESSOR} )

