From 2fbd6e48b3f7575f3618d13a46b440aceddd7546 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sat, 2 May 2026 15:00:27 +0800
Subject: [PATCH] Update netsource.c

---
 lib/src/phy/io/netsource.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/src/phy/io/netsource.c b/lib/src/phy/io/netsource.c
index b772af0b9cb..d50e56b5df3 100644
--- a/lib/src/phy/io/netsource.c
+++ b/lib/src/phy/io/netsource.c
@@ -27,6 +27,7 @@
 #include <stdlib.h>
 #include <strings.h>
 #include <sys/socket.h>
+#include <sys/time.h>
 #include <unistd.h>
 
 #include "srsran/phy/io/netsource.h"
--- a/lib/src/phy/io/netsink.c
+++ b/lib/src/phy/io/netsink.c
@@ -84,7 +84,7 @@ int srsran_netsink_set_nonblocking(srsran_netsink_t* q)
 int srsran_netsink_write(srsran_netsink_t* q, void* buffer, int nof_bytes)
 {
   if (!q->connected) {
-    if (connect(q->sockfd, &q->servaddr, sizeof(q->servaddr)) < 0) {
+    if (connect(q->sockfd, (struct sockaddr *)&q->servaddr, sizeof(q->servaddr)) < 0) {
       if (errno == ECONNREFUSED || errno == EINPROGRESS) {
         return SRSRAN_SUCCESS;
       } else {
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,7 +30,7 @@ endif(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
 ########################################################################
 # Project setup
 ########################################################################
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 4.0)
 project( SRSRAN )
 message( STATUS "CMAKE_SYSTEM: " ${CMAKE_SYSTEM} )
 message( STATUS "CMAKE_SYSTEM_PROCESSOR: " ${CMAKE_SYSTEM_PROCESSOR} )
From 88d88dbd696071ad413769d061ca8770a8ebc624 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sat, 2 May 2026 06:15:13 +0800
Subject: [PATCH] Add Boost version 1.89 to CMakeLists

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c2ec9ac4585..2f2316a4066 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -276,6 +276,7 @@ set(Boost_ADDITIONAL_VERSIONS
     "1.55.0" "1.55" "1.56.0" "1.56" "1.57.0" "1.57" "1.58.0" "1.58" "1.59.0" "1.59"
     "1.60.0" "1.60" "1.61.0" "1.61" "1.62.0" "1.62" "1.63.0" "1.63" "1.64.0" "1.64"
     "1.65.0" "1.65" "1.66.0" "1.66" "1.67.0" "1.67" "1.68.0" "1.68" "1.69.0" "1.69"
+    "1.89.0" "1.89"
 )
 find_package(Boost "1.35" COMPONENTS ${BOOST_REQUIRED_COMPONENTS})
 
