From 5543cf0c7347e212bf7ea9f8c4a678c5061697c2 Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Sun, 26 Jul 2026 01:52:24 +0800
Subject: [PATCH] Update restreamer.js

---
 src/utils/restreamer.js | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/src/utils/restreamer.js b/src/utils/restreamer.js
index e31bab7a..9aceaa07 100644
--- a/src/utils/restreamer.js
+++ b/src/utils/restreamer.js
@@ -733,22 +733,22 @@ class Restreamer {
 					rtmp: {
 						enabled: false,
 						secure: false,
-						host: '',
-						local: 'localhost',
+						host: '192.168.1.1',
+						local: '',
 						app: '',
 						token: '',
 					},
 					srt: {
 						enabled: false,
-						host: '',
-						local: 'localhost',
+						host: '192.168.1.1',
+						local: '',
 						token: '',
 						passphrase: '',
 					},
 					hls: {
 						secure: false,
-						host: '',
-						local: 'localhost',
+						host: '192.168.1.1',
+						local: '',
 						credentials: '',
 					},
 				},
@@ -782,7 +782,7 @@ class Restreamer {
 		}
 
 		const isIP = (host) => {
-			if (host === 'localhost') {
+			if (host === '192.168.1.1') {
 				return true;
 			}
 
@@ -862,7 +862,7 @@ class Restreamer {
 		// This is used for FFmpeg to access the HLS stream. This will happen always via HTTP.
 		// If the HTTP server is bound to a specific address, we'll use this one, localhost otherwise.
 		let [http_host, http_port] = splitHostPort(val.config.address);
-		config.source.network.hls.local = http_host.length !== 0 ? http_host : 'localhost';
+		config.source.network.hls.local = http_host.length !== 0 ? http_host : '192.168.1.1';
 		if (http_port !== '80') {
 			config.source.network.hls.local += ':' + http_port;
 		}
@@ -895,7 +895,7 @@ class Restreamer {
 		// This is used for FFmpeg to access the RTMP stream. If the RTMP server is bound to a
 		// specific address, we'll use this one, localhost otherwise.
 		const [rtmp_host, rtmp_port] = splitHostPort(val.config.rtmp.address);
-		config.source.network.rtmp.local = rtmp_host.length !== 0 ? rtmp_host : 'localhost';
+		config.source.network.rtmp.local = rtmp_host.length !== 0 ? rtmp_host : '192.168.1.1';
 		if (rtmp_port !== '1935') {
 			config.source.network.rtmp.host += ':' + rtmp_port;
 			config.source.network.rtmp.local += ':' + rtmp_port;
@@ -917,7 +917,7 @@ class Restreamer {
 		config.source.network.srt.host = config.hostname;
 
 		let [srt_host, srt_port] = splitHostPort(val.config.srt.address);
-		config.source.network.srt.local = srt_host.length !== 0 ? srt_host : 'localhost';
+		config.source.network.srt.local = srt_host.length !== 0 ? srt_host : '192.168.1.1';
 		config.source.network.srt.host += ':' + srt_port;
 		config.source.network.srt.local += ':' + srt_port;
 
@@ -1668,7 +1668,7 @@ class Restreamer {
 			rtmp_enabled = true;
 		}
 		if (
-			proc.input[0].address.includes('rtmp://localhost') &&
+			proc.input[0].address.includes('rtmp://192.168.1.1') &&
 			proc.input[0].address.includes(channel.channelid) &&
 			!proc.input[0].address.includes('ingest')
 		) {
@@ -1683,7 +1683,7 @@ class Restreamer {
 			srt_enabled = true;
 		}
 		if (
-			proc.input[0].address.includes('srt://localhost') &&
+			proc.input[0].address.includes('srt://192.168.1.1') &&
 			proc.input[0].address.includes(channel.channelid) &&
 			!proc.input[0].address.includes('ingest')
 		) {
From 246147579ac4ce1fcae44ecc1f3991a87d029a27 Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Sun, 26 Jul 2026 01:49:08 +0800
Subject: [PATCH] Update Network.js

---
 src/views/Edit/Sources/Network.js | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/views/Edit/Sources/Network.js b/src/views/Edit/Sources/Network.js
index dd5e1120..533ff581 100644
--- a/src/views/Edit/Sources/Network.js
+++ b/src/views/Edit/Sources/Network.js
@@ -123,8 +123,8 @@ const initConfig = (initialConfig) => {
 	config.rtmp = {
 		enabled: false,
 		secure: false,
-		host: 'localhost',
-		local: 'localhost',
+		host: '192.168.1.1',
+		local: '',
 		app: '',
 		token: '',
 		...config.rtmp,
@@ -132,8 +132,8 @@ const initConfig = (initialConfig) => {
 
 	config.srt = {
 		enabled: false,
-		host: 'localhost',
-		local: 'localhost',
+		host: '192.168.1.1',
+		local: '',
 		token: '',
 		passphrase: '',
 		...config.srt,
@@ -141,8 +141,8 @@ const initConfig = (initialConfig) => {
 
 	config.hls = {
 		secure: false,
-		host: 'localhost',
-		local: 'localhost',
+		host: '192.168.1.1',
+		local: '',
 		credentials: '',
 		...config.hls,
 	};
From dc716782910e6aa9a9ef7369c84770c1c9b56474 Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Sun, 26 Jul 2026 01:45:13 +0800
Subject: [PATCH] Change localhost check to specific IP address

---
 src/utils/anonymizer.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/anonymizer.js b/src/utils/anonymizer.js
index b6be6ed9..387ccd25 100644
--- a/src/utils/anonymizer.js
+++ b/src/utils/anonymizer.js
@@ -3,7 +3,7 @@ import urlparser from 'url-parse';
 const anonymize_url = (url) => {
 	let u = urlparser(url, true);
 
-	if (u.hostname !== 'localhost') {
+	if (u.hostname !== '192.168.1.1') {
 		return `${u.protocol}//[anonymized]`;
 	}
 
