From a0bdceff082cd945512064f79a11587dacdc4589 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sat, 23 Aug 2025 10:27:34 +0800
Subject: [PATCH] Update Makefile.PL

---
 Makefile.PL | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 8a8ef1c..35dbfdc 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -13,16 +13,6 @@ my %MakeParams = InitMakeParams();
 WriteMakefile(%MakeParams);
 
 
-sub MY::postamble {	
-	"
-LibIDN.pm: _LibIDN.pm _test.pl
-	perl Makefile.PL $options
-
-test.pl: _test.pl
-	perl Makefile.PL $options
-"
-}
-
 sub InitMakeParams
 {
 	my %Params =
