From c804b1fc30dc5a31557fe4d967067b426045b47f Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sat, 23 Aug 2025 09:52:37 +0800
Subject: [PATCH] Update Makefile.PL

---
 Makefile.PL | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 6709fe1..8a8ef1c 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -49,7 +49,7 @@ sub InitMakeParams
 
 	if ($libdir)
 	{
-		$Params{LIBS} = "-L$libdir -lidn";
+		$Params{LIBS} = "-L@STAGING_DIR@/lib -lidn";
 	}
 	else
 	{
@@ -58,7 +58,7 @@ sub InitMakeParams
 
 	if ($incdir)
 	{
-		$Params{INC} = "-I$incdir";
+		$Params{INC} = "-I@STAGING_DIR@/include";
 	}
 	else
 	{
