From 846a76bdcac33463c9d2aacb998e4f58c8a3dfd2 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Fri, 3 Apr 2026 00:47:15 +0800
Subject: [PATCH] Change FATAL_ERROR messages to STATUS in CMakeLists

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 41e77c2efc3..60d5f1ed79d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,7 +145,7 @@ target_include_directories (H5match_types PRIVATE "${HDF5_SRC_BINARY_DIR};${HDF5
 # check if a pre-generated H5fortran_types.F90 file is present
 if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
   if (CMAKE_CROSSCOMPILING AND NOT CMAKE_CROSSCOMPILING_EMULATOR)
-    message (FATAL_ERROR "No pre-generated H5fortran_types.F90 file found - cannot generate without cross-compiling emulator")
+    message (STATUS "No pre-generated H5fortran_types.F90 file found - cannot generate without cross-compiling emulator")
   endif ()
   message (STATUS "No pre-generated H5fortran_types.F90 file found - generating one")
   # execute the H5match_types program
@@ -330,7 +330,7 @@ set (f90_F_GEN_SOURCES
 # check if a pre-generated H5fortran_types.F90 file is present
 if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5_gen.F90")
   if (CMAKE_CROSSCOMPILING AND NOT CMAKE_CROSSCOMPILING_EMULATOR)
-    message (FATAL_ERROR "No pre-generated H5_gen.F90 file found - cannot generate without cross-compiling emulator")
+    message (STATUS "No pre-generated H5_gen.F90 file found - cannot generate without cross-compiling emulator")
   endif ()
   message (STATUS "No pre-generated H5_gen.F90 file found - generating one")
   add_custom_command (TARGET H5_buildiface POST_BUILD
