From 99f0a45def5d3e546080f4f7625f6e0f563ff387 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Wed, 3 Dec 2025 11:49:53 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 fortran/test/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fortran/test/CMakeLists.txt b/fortran/test/CMakeLists.txt
index 5c34919047f..e59a4b8f380 100644
--- a/fortran/test/CMakeLists.txt
+++ b/fortran/test/CMakeLists.txt
@@ -108,7 +108,7 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/tf_gen.F90")
     add_custom_command (TARGET H5_test_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_FORTRAN_TESTS_BINARY_DIR}/tf_gen.F90
         COMMAND    ${CMAKE_COMMAND}
-        ARGS       -E env ${CROSSCOMPILING_PATH}
+        ARGS       -E ${CROSSCOMPILING_PATH}
         COMMAND    $<TARGET_FILE:H5_test_buildiface>
         WORKING_DIRECTORY ${HDF5_FORTRAN_TESTS_BINARY_DIR}
         COMMENT "Generating the tf_gen.F90 static file"
