--- a/hl/fortran/src/CMakeLists.txt
+++ b/hl/fortran/src/CMakeLists.txt
@@ -127,6 +127,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5LTff_gen.F90")
   if (CMAKE_CROSSCOMPILING)
     add_custom_command (TARGET H5HL_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_HL_F90_BINARY_DIR}/H5LTff_gen.F90 ${HDF5_HL_F90_BINARY_DIR}/H5TBff_gen.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5HL_buildiface
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5HL_buildiface1 H5HL_buildiface
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E env ${CROSSCOMPILING_PATH}
         COMMAND    $<TARGET_FILE:H5HL_buildiface>
@@ -136,6 +138,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5LTff_gen.F90")
   else ()
     add_custom_command (TARGET H5HL_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_HL_F90_BINARY_DIR}/H5LTff_gen.F90 ${HDF5_HL_F90_BINARY_DIR}/H5TBff_gen.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5HL_buildiface
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5HL_buildiface1 H5HL_buildiface
         COMMAND    $<TARGET_FILE:H5HL_buildiface>
         WORKING_DIRECTORY ${HDF5_HL_F90_BINARY_DIR}
         COMMENT "Generating the H5LTff_gen.F90, H5TBff_gen.F90 files"
--- a/fortran/src/CMakeLists.txt
+++ b/fortran/src/CMakeLists.txt
@@ -149,6 +149,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
   if (CMAKE_CROSSCOMPILING)
     add_custom_command (TARGET H5match_types POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5f90i_gen.h ${HDF5_F90_BINARY_DIR}/H5fortran_types.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5match_types
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5match_types1 H5match_types
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E env ${CROSSCOMPILING_PATH}
         COMMAND    $<TARGET_FILE:H5match_types>
@@ -158,6 +160,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
   else ()
     add_custom_command (TARGET H5match_types POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5f90i_gen.h ${HDF5_F90_BINARY_DIR}/H5fortran_types.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5match_types
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5match_types1 H5match_types
         COMMAND    $<TARGET_FILE:H5match_types>
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5fortran_types.F90 and H5f90i_gen.h files"
@@ -341,6 +345,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5_gen.F90")
   if (CMAKE_CROSSCOMPILING)
     add_custom_command (TARGET H5_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5_gen.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5_buildiface
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5_buildiface1 H5_buildiface
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E env ${CROSSCOMPILING_PATH}
         COMMAND    $<TARGET_FILE:H5_buildiface>
@@ -350,6 +356,8 @@ if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5_gen.F90")
   else ()
     add_custom_command (TARGET H5_buildiface POST_BUILD
         BYPRODUCTS ${HDF5_F90_BINARY_DIR}/H5_gen.F90
+        COMMAND    ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/bin/H5_buildiface
+        COMMAND    ${CMAKE_COMMAND} -E rename ${CMAKE_BINARY_DIR}/bin/H5_buildiface1 H5_buildiface
         COMMAND    $<TARGET_FILE:H5_buildiface>
         WORKING_DIRECTORY ${HDF5_F90_BINARY_DIR}
         COMMENT "Generating the H5_gen.F90 file"
