From 9ee9c6ea8137ea2a9718b0a31d0b965806868cf1 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 6 Dec 2025 06:40:38 +0800
Subject: [PATCH] Update H5match_types.c

---
 fortran/src/H5match_types.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/fortran/src/H5match_types.c b/fortran/src/H5match_types.c
index 40d21e97cc0..3ed1ce9335a 100644
--- a/fortran/src/H5match_types.c
+++ b/fortran/src/H5match_types.c
@@ -27,6 +27,8 @@
  ******
  */
 
+#include <stdint.h>
+#include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <assert.h>
--- a/configure
+++ b/configure
@@ -7567,27 +7567,6 @@ main (void)
 }
 
 _ACEOF
-        if test "$cross_compiling" = yes
-then :
-  { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else $as_nop
-  if ac_fn_c_try_run "$LINENO"
-then :
-
-            MY_LDBL_DIG=$(./conftest$EXEEXT 2>&1 | sed -n '1p')
-            MY_FLT128_DIG=$(./conftest$EXEEXT 2>&1 | sed -n '2p')
-
-else $as_nop
-
-            as_fn_error $? "C program fails to build or run!" "$LINENO" 5
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam
-fi
 
 
 
@@ -9173,76 +9152,6 @@ ac_link='$FC -o conftest$ac_exeext $FCFLAGS $LDFLAGS $ac_fcflags_srcext conftest
 ac_compiler_gnu=$ac_cv_fc_compiler_gnu
 
 TEST_SRC="`sed -n '/PROGRAM FC_AVAIL_KINDS/,/END PROGRAM FC_AVAIL_KINDS/p' $srcdir/m4/aclocal_fc.f90`"
-if test "$cross_compiling" = yes
-then :
-  { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else $as_nop
-  cat > conftest.$ac_ext <<_ACEOF
-$TEST_SRC
-_ACEOF
-if ac_fn_fc_try_run "$LINENO"
-then :
-
-
-        pac_validIntKinds=$(./conftest$EXEEXT 2>&1 | sed -n '1p')
-        pac_validRealKinds=$(./conftest$EXEEXT 2>&1 | sed -n '2p')
-        PAC_FC_MAX_REAL_PRECISION=$(./conftest$EXEEXT 2>&1 | sed -n '3p')
-
-printf "%s\n" "#define PAC_FC_MAX_REAL_PRECISION $PAC_FC_MAX_REAL_PRECISION" >>confdefs.h
-
-
-        PAC_FC_ALL_INTEGER_KINDS="{`echo $pac_validIntKinds`}"
-        PAC_FC_ALL_REAL_KINDS="{`echo $pac_validRealKinds`}"
-
-        PAC_FORTRAN_NUM_INTEGER_KINDS=$(./conftest$EXEEXT 2>&1 | sed -n '4p')
-        H5CONFIG_F_NUM_IKIND="INTEGER, PARAMETER :: num_ikinds = `echo $PAC_FORTRAN_NUM_INTEGER_KINDS`"
-        H5CONFIG_F_IKIND="INTEGER, DIMENSION(1:num_ikinds) :: ikind = (/`echo $pac_validIntKinds`/)"
-        H5CONFIG_F_NUM_RKIND="INTEGER, PARAMETER :: num_rkinds = $(./conftest$EXEEXT 2>&1 | sed -n '5p')"
-        H5CONFIG_F_RKIND="INTEGER, DIMENSION(1:num_rkinds) :: rkind = (/`echo $pac_validRealKinds`/)"
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_RKIND $H5CONFIG_F_NUM_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_IKIND $H5CONFIG_F_NUM_IKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_RKIND $H5CONFIG_F_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_IKIND $H5CONFIG_F_IKIND" >>confdefs.h
-
-
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Number of Fortran INTEGER KINDs" >&5
-printf %s "checking for Number of Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FORTRAN_NUM_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FORTRAN_NUM_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran INTEGER KINDs" >&5
-printf %s "checking for Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REAL KINDs" >&5
-printf %s "checking for Fortran REAL KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_REAL_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_REAL_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REALs maximum decimal precision" >&5
-printf %s "checking for Fortran REALs maximum decimal precision... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_MAX_REAL_PRECISION" >&5
-printf "%s\n" "$PAC_FC_MAX_REAL_PRECISION" >&6; }
-
-else $as_nop
-
-    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: Error" >&5
-printf "%s\n" "Error" >&6; }
-    as_fn_error $? "Failed to run Fortran program to determine available KINDs" "$LINENO" 5
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
 
 
 ac_ext=${ac_fc_srcext-f}
@@ -9328,88 +9237,6 @@ ac_link='$FC -o conftest$ac_exeext $FCFLAGS $LDFLAGS $ac_fcflags_srcext conftest
 ac_compiler_gnu=$ac_cv_fc_compiler_gnu
 
 TEST_SRC="`sed -n '/PROGRAM FC08_AVAIL_KINDS/,/END PROGRAM FC08_AVAIL_KINDS/p' $srcdir/m4/aclocal_fc.f90`"
-if test "$cross_compiling" = yes
-then :
-  { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else $as_nop
-  cat > conftest.$ac_ext <<_ACEOF
-$TEST_SRC
-_ACEOF
-if ac_fn_fc_try_run "$LINENO"
-then :
-
-
-        pac_validIntKinds=$(./conftest$EXEEXT 2>&1 | sed -n '1p')
-        pac_validRealKinds=$(./conftest$EXEEXT 2>&1 | sed -n '2p')
-        PAC_FC_MAX_REAL_PRECISION=$(./conftest$EXEEXT 2>&1 | sed -n '3p')
-
-printf "%s\n" "#define PAC_FC_MAX_REAL_PRECISION $PAC_FC_MAX_REAL_PRECISION" >>confdefs.h
-
-
-        PAC_FC_ALL_INTEGER_KINDS="{`echo $pac_validIntKinds`}"
-        PAC_FC_ALL_REAL_KINDS="{`echo $pac_validRealKinds`}"
-
-        PAC_FORTRAN_NUM_INTEGER_KINDS=$(./conftest$EXEEXT 2>&1 | sed -n '4p')
-        H5CONFIG_F_NUM_IKIND="INTEGER, PARAMETER :: num_ikinds = `echo $PAC_FORTRAN_NUM_INTEGER_KINDS`"
-        H5CONFIG_F_IKIND="INTEGER, DIMENSION(1:num_ikinds) :: ikind = (/`echo $pac_validIntKinds`/)"
-        H5CONFIG_F_NUM_RKIND="INTEGER, PARAMETER :: num_rkinds = $(./conftest$EXEEXT 2>&1 | sed -n '5p')"
-        H5CONFIG_F_RKIND="INTEGER, DIMENSION(1:num_rkinds) :: rkind = (/`echo $pac_validRealKinds`/)"
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_RKIND $H5CONFIG_F_NUM_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_IKIND $H5CONFIG_F_NUM_IKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_RKIND $H5CONFIG_F_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_IKIND $H5CONFIG_F_IKIND" >>confdefs.h
-
-
-        PAC_FORTRAN_NUM_LOGICAL_KINDS=$(./conftest$EXEEXT 2>&1 | sed -n '6p')
-        pac_validLogicalKinds=$(./conftest$EXEEXT 2>&1 | sed -n '7p')
-        PAC_FC_ALL_LOGICAL_KINDS="{`echo $pac_validLogicalKinds`}"
-
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Number of Fortran INTEGER KINDs" >&5
-printf %s "checking for Number of Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FORTRAN_NUM_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FORTRAN_NUM_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran INTEGER KINDs" >&5
-printf %s "checking for Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REAL KINDs" >&5
-printf %s "checking for Fortran REAL KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_REAL_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_REAL_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REALs maximum decimal precision" >&5
-printf %s "checking for Fortran REALs maximum decimal precision... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_MAX_REAL_PRECISION" >&5
-printf "%s\n" "$PAC_FC_MAX_REAL_PRECISION" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Number of Fortran LOGICAL KINDs" >&5
-printf %s "checking for Number of Fortran LOGICAL KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FORTRAN_NUM_LOGICAL_KINDS" >&5
-printf "%s\n" "$PAC_FORTRAN_NUM_LOGICAL_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran LOGICAL KINDs" >&5
-printf %s "checking for Fortran LOGICAL KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_LOGICAL_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_LOGICAL_KINDS" >&6; }
-
-else $as_nop
-
-    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: Error" >&5
-printf "%s\n" "Error" >&6; }
-    as_fn_error $? "Failed to run Fortran program to determine available KINDs" "$LINENO" 5
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
 
 ac_ext=${ac_fc_srcext-f}
 ac_compile='$FC -c $FCFLAGS $ac_fcflags_srcext conftest.$ac_ext >&5'
@@ -9425,76 +9252,6 @@ ac_link='$FC -o conftest$ac_exeext $FCFLAGS $LDFLAGS $ac_fcflags_srcext conftest
 ac_compiler_gnu=$ac_cv_fc_compiler_gnu
 
 TEST_SRC="`sed -n '/PROGRAM FC_AVAIL_KINDS/,/END PROGRAM FC_AVAIL_KINDS/p' $srcdir/m4/aclocal_fc.f90`"
-if test "$cross_compiling" = yes
-then :
-  { { printf "%s\n" "$as_me:${as_lineno-$LINENO}: error: in \`$ac_pwd':" >&5
-printf "%s\n" "$as_me: error: in \`$ac_pwd':" >&2;}
-as_fn_error $? "cannot run test program while cross compiling
-See \`config.log' for more details" "$LINENO" 5; }
-else $as_nop
-  cat > conftest.$ac_ext <<_ACEOF
-$TEST_SRC
-_ACEOF
-if ac_fn_fc_try_run "$LINENO"
-then :
-
-
-        pac_validIntKinds=$(./conftest$EXEEXT 2>&1 | sed -n '1p')
-        pac_validRealKinds=$(./conftest$EXEEXT 2>&1 | sed -n '2p')
-        PAC_FC_MAX_REAL_PRECISION=$(./conftest$EXEEXT 2>&1 | sed -n '3p')
-
-printf "%s\n" "#define PAC_FC_MAX_REAL_PRECISION $PAC_FC_MAX_REAL_PRECISION" >>confdefs.h
-
-
-        PAC_FC_ALL_INTEGER_KINDS="{`echo $pac_validIntKinds`}"
-        PAC_FC_ALL_REAL_KINDS="{`echo $pac_validRealKinds`}"
-
-        PAC_FORTRAN_NUM_INTEGER_KINDS=$(./conftest$EXEEXT 2>&1 | sed -n '4p')
-        H5CONFIG_F_NUM_IKIND="INTEGER, PARAMETER :: num_ikinds = `echo $PAC_FORTRAN_NUM_INTEGER_KINDS`"
-        H5CONFIG_F_IKIND="INTEGER, DIMENSION(1:num_ikinds) :: ikind = (/`echo $pac_validIntKinds`/)"
-        H5CONFIG_F_NUM_RKIND="INTEGER, PARAMETER :: num_rkinds = $(./conftest$EXEEXT 2>&1 | sed -n '5p')"
-        H5CONFIG_F_RKIND="INTEGER, DIMENSION(1:num_rkinds) :: rkind = (/`echo $pac_validRealKinds`/)"
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_RKIND $H5CONFIG_F_NUM_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_NUM_IKIND $H5CONFIG_F_NUM_IKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_RKIND $H5CONFIG_F_RKIND" >>confdefs.h
-
-
-printf "%s\n" "#define H5CONFIG_F_IKIND $H5CONFIG_F_IKIND" >>confdefs.h
-
-
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Number of Fortran INTEGER KINDs" >&5
-printf %s "checking for Number of Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FORTRAN_NUM_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FORTRAN_NUM_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran INTEGER KINDs" >&5
-printf %s "checking for Fortran INTEGER KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_INTEGER_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_INTEGER_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REAL KINDs" >&5
-printf %s "checking for Fortran REAL KINDs... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_ALL_REAL_KINDS" >&5
-printf "%s\n" "$PAC_FC_ALL_REAL_KINDS" >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: checking for Fortran REALs maximum decimal precision" >&5
-printf %s "checking for Fortran REALs maximum decimal precision... " >&6; }
-        { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: $PAC_FC_MAX_REAL_PRECISION" >&5
-printf "%s\n" "$PAC_FC_MAX_REAL_PRECISION" >&6; }
-
-else $as_nop
-
-    { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: Error" >&5
-printf "%s\n" "Error" >&6; }
-    as_fn_error $? "Failed to run Fortran program to determine available KINDs" "$LINENO" 5
-
-fi
-rm -f core *.core core.conftest.* gmon.out bb.out conftest$ac_exeext \
-  conftest.$ac_objext conftest.beam conftest.$ac_ext
-fi
 
 
 ac_ext=${ac_fc_srcext-f}
