From 4eb5132e606160122223a2c17ca40aca6f517abe Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 3 Dec 2025 13:40:05 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 fortran/src/CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/fortran/src/CMakeLists.txt b/fortran/src/CMakeLists.txt
index d84143b00f0..49354fdc5ea 100644
--- a/fortran/src/CMakeLists.txt
+++ b/fortran/src/CMakeLists.txt
@@ -141,6 +141,14 @@ add_executable (H5match_types
     ${HDF5_F90_BINARY_DIR}/H5fort_type_defines.h
     ${HDF5_F90_SRC_SOURCE_DIR}/H5match_types.c
 )
+
+if(DEFINED H5MATCH_TYPES_C_COMPILER)
+	set(CMAKE_C_COMPILER "${H5MATCH_TYPES_C_COMPILER}")
+	set(CMAKE_C_FLAGS "")
+	set(CMAKE_EXE_LINKER_FLAGS "")
+	set(CMAKE_SYSROOT "")
+endif()
+
 target_include_directories (H5match_types PRIVATE "${HDF5_SRC_BINARY_DIR};${HDF5_SRC_INCLUDE_DIRS};${HDF5_F90_BINARY_DIR};$<$<BOOL:${HDF5_ENABLE_PARALLEL}>:${MPI_C_INCLUDE_DIRS}>")
 # check if a pre-generated H5fortran_types.F90 file is present
 if (NOT EXISTS "${HDF5_GENERATED_SOURCE_DIR}/H5fortran_types.F90")
