From 5de4c1fb9e7e52bb250b5ed1fc65e8c180cb1d7b Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Mon, 1 Jun 2026 08:20:27 +0800
Subject: [PATCH] Update tnode.h

---
 deps/v8/src/codegen/tnode.h | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/deps/v8/src/codegen/tnode.h b/deps/v8/src/codegen/tnode.h
index 83a05d0d441c29..f2407e57f14479 100644
--- a/deps/v8/src/codegen/tnode.h
+++ b/deps/v8/src/codegen/tnode.h
@@ -6,6 +6,26 @@
 #define V8_CODEGEN_TNODE_H_
 
 #include "src/codegen/machine-type.h"
+#include "src/objects/cell.h"
+#include "src/objects/js-generator.h"
+#include "src/objects/js-promise.h"
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
 
 namespace v8 {
 namespace internal {
