--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ ARGON2_VERSION ?= ZERO
 # Note; if Linux and not Debian/Ubuntu version also add lib override to make command-line
 #       for RedHat/Fedora, add: LIBRARY_REL=lib64
 DESTDIR ?=
-PREFIX ?= /usr
+PREFIX ?=
 
 # Increment on an ABI breaking change
 ABI_VERSION = 1
--- a/libargon2.pc.in
+++ b/libargon2.pc.in
@@ -7,8 +7,8 @@
 
 prefix=@PREFIX@
 exec_prefix=${prefix}
-libdir=${prefix}/@HOST_MULTIARCH@
-includedir=${prefix}/@INCLUDE@
+libdir=${prefix}/lib
+includedir=${prefix}/include
 
 Name: libargon2
 Description: Development libraries for libargon2
