From 16c052a8683eb4531b36e241148ca4f3b8c0b237 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Fri, 10 Apr 2026 11:35:53 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index b21c51a..23da000 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,7 +1,6 @@
 [build-system]
 requires = [
     "hatchling>=1.5.0",
-    "jupyterlab>=4.0.0,<5",
     "hatch-nodejs-version>=0.3.2",
 ]
 build-backend = "hatchling.build"
