From 9f642b6105975401d76b54fbad80cdfacbbe3236 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Mon, 26 Jan 2026 12:04:18 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 CMakeLists.txt | 18 +++++++++++++-----
 1 file changed, 13 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f6bfc52d160..9a9ef80186b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,10 +87,18 @@ if(NOT CPU_VARIANTS)
     set(CPU_VARIANTS "ggml-cpu")
 endif()
 
-install(TARGETS ggml-base ${CPU_VARIANTS}
-    RUNTIME_DEPENDENCIES
-        PRE_EXCLUDE_REGEXES ".*"
-    RUNTIME DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
+install(TARGETS ggml-cpu
+#    RUNTIME_DEPENDENCIES
+#        PRE_EXCLUDE_REGEXES ".*"
+#    RUNTIME DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
+    LIBRARY DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
+    FRAMEWORK DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
+)
+
+install(TARGETS ggml-base
+#    RUNTIME_DEPENDENCIES
+#        PRE_EXCLUDE_REGEXES ".*"
+#    RUNTIME DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
     LIBRARY DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
     FRAMEWORK DESTINATION ${OLLAMA_INSTALL_DIR} COMPONENT CPU
 )
@@ -216,4 +224,4 @@ if(MLX_ENGINE)
                 COMPONENT MLX)
         endif()
     endif()
-endif()
\ No newline at end of file
+endif()
