From 9337a022d159be1c2a34524e75ee3a52c2afab6c Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 4 Apr 2026 22:16:37 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2ea1778..e48de16 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -83,9 +83,6 @@ compile = [
 ]
 dist = []
 install = []
-setup = [
-    "--vsenv",  # Force use of MSVC on Windows, ignored on other platforms
-]
 
 
 [tool.cibuildwheel]
