From 91a46f1b5165ea4a1026bceb4260157325b96a4d Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Thu, 5 Feb 2026 19:06:15 +0800
Subject: [PATCH] Update Configure.pm

---
 Configure.pm | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/Configure.pm b/Configure.pm
index 82aab81..f48cb46 100644
--- a/Configure.pm
+++ b/Configure.pm
@@ -171,8 +171,8 @@ sub Compile { # Feed code to compiler. On error, return status and text
 	print F $code;
 	close(F);
 	print "Compiling |$code|\n"  if $Verbose;
-	my($result) = scalar(`$C_cc $C_ccflags -c $in $C_ldflags $C_libs $options 2>&1`);
-	print "Executing '$C_cc $C_ccflags -c $in $C_ldflags $C_libs $options 2>&1'\n"  if $Verbose;
+	my($result) = scalar(`$C_cc $C_ccflags -c $in $C_libs $options 2>&1`);
+	print "Executing '$C_cc $C_ccflags -c $in $C_libs $options 2>&1'\n"  if $Verbose;
 	my($error) = $?;
         my($error2) = ! -e $out;
 	unlink($in,$out);
@@ -208,8 +208,8 @@ sub Link { # Feed code to compiler and linker. On error, return status and text
 	print F $code;
 	close(F);
 	print "Linking |$code|\n" if $Verbose;
-	my($result) = scalar(`$C_cc $C_ccflags -o $file $in $C_ldflags $C_libs $options 2>&1`);
-	print "Executing '$C_cc $C_ccflags -o $file $in $C_ldflags $C_libs $options 2>&1'\n" if $Verbose;
+	my($result) = scalar(`$C_cc $C_ccflags -o $file $in $C_libs $options 2>&1`);
+	print "Executing '$C_cc $C_ccflags -o $file $in $C_libs $options 2>&1'\n" if $Verbose;
 	my($error)=$?;
 	print "Error linking: $error, |$result|\n" if $Verbose;
 	unlink($in,$out,$file);
@@ -248,8 +248,8 @@ sub Execute { #Compile, link, and execute.
 	print F $code;
 	close(F);
 	print "Executing |$code|\n" if $Verbose;
-	my($result) = scalar(`$C_cc $C_ccflags -o $file $in $C_ldflags $C_libs $options 2>&1`);
-	print "Executing '$C_cc $C_ccflags -o $file $in $C_ldflags $C_libs $options 2>&1'\n" if $Verbose;
+	my($result) = scalar(`$C_cc $C_ccflags -o $file $in $C_libs $options 2>&1`);
+	print "Executing '$C_cc $C_ccflags -o $file $in $C_libs $options 2>&1'\n" if $Verbose;
 	my($error) = $?;
 	unlink($in,$out);
 	if(!$error) {
