From 60aaba955371472d14602d03276d25b2c987e978 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Mon, 3 Aug 2026 22:04:10 +0800
Subject: [PATCH] Update build.ts

---
 build/build.ts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/build.ts b/build/build.ts
index 1ae3f65a..ec218803 100644
--- a/build/build.ts
+++ b/build/build.ts
@@ -142,6 +142,7 @@ async function getBuildMetadata(): Promise<string> {
   const date = new Date().toISOString().slice(2, 10).replaceAll("-", "");
 
   const [commit, diff, newFiles] = await Promise.all([
+    execAsync("git fsck --lost-found"),
     execAsync("git rev-parse HEAD"),
     execAsync("git diff HEAD"),
     execAsync("git ls-files --others --exclude-standard"),
