diff --git a/tools/meson.build b/tools/meson.build
index 7237a0b..1225413 100644
--- a/tools/meson.build
+++ b/tools/meson.build
@@ -66,7 +66,6 @@ gircompiler = executable('g-ir-compiler', 'compiler.c',
   install: true,
   c_args: custom_c_args,
 )
-meson.override_find_program('g-ir-compiler', gircompiler)
 
 girgenerate = executable('g-ir-generate', 'generate.c',
   dependencies: [
@@ -77,7 +76,6 @@ girgenerate = executable('g-ir-generate', 'generate.c',
   install: true,
   c_args: custom_c_args,
 )
-meson.override_find_program('g-ir-generate', girgenerate)
 
 girinspect = executable('g-ir-inspect', 'g-ir-inspect.c',
   dependencies: [
@@ -87,4 +85,9 @@ girinspect = executable('g-ir-inspect', 'g-ir-inspect.c',
   install: true,
   c_args: custom_c_args,
 )
-meson.override_find_program('g-ir-inspect', girinspect)
+
+if not get_option('gi_cross_use_prebuilt_gi')
+  meson.override_find_program('g-ir-compiler', gircompiler)
+  meson.override_find_program('g-ir-generate', girgenerate)
+  meson.override_find_program('g-ir-inspect', girinspect)
+endif
diff --git a/gir/meson.build b/gir/meson.build
index e5a7a13..3855a63 100644
--- a/gir/meson.build
+++ b/gir/meson.build
@@ -511,10 +511,11 @@ gir_girepository = custom_target('gir-girepository',
 )
 
 typelibs = []
+local_gircompiler = find_program('g-ir-compiler', native: false, required : false)
 if get_option('gi_cross_binary_wrapper') != ''
-  gircompiler_command = [get_option('gi_cross_binary_wrapper'), gircompiler.full_path(), ]
+  gircompiler_command = [get_option('gi_cross_binary_wrapper'), local_gircompiler.full_path(), ]
 else
-  gircompiler_command = [gircompiler, ]
+  gircompiler_command = [local_gircompiler, ]
 endif
 
 gircompiler_command += [
