From bbc25d787f6ebdca319777d1ba566f44d3e78b5d Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 26 Apr 2026 21:02:12 +0800
Subject: [PATCH] Update stun.c

---
 src/stun.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/stun.c b/src/stun.c
index 523c19b..8ad9529 100644
--- a/src/stun.c
+++ b/src/stun.c
@@ -67,6 +67,12 @@
  *
  */
 
+#include <cassert>
+#include <cstring>
+#include <iostream>
+#include <cstdlib>
+#include <errno.h>
+
 #ifdef HAVE_CONFIG_H
 #include "ortp-config.h"
 #endif
