From 331f6a6ad3a2e81e27266ebaf027b562083d94b9 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 15 Nov 2025 12:48:04 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyproject.toml b/pyproject.toml
index ed410fb0f..c90810d7c 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,6 @@
+[tool.setuptools]
+packages = []
+
 [tool.black]
 target-version = ["py311"]
 include = 'custom_components/localtuya/.*\.py'
