From 6a495dd93af59d582a79e2496cdf0e1f3e864d67 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 18 Feb 2025 23:21:09 +0800
Subject: [PATCH] Update compute.h

---
 src/xnnpack/compute.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xnnpack/compute.h b/src/xnnpack/compute.h
index ec72466..1bc492b 100644
--- a/src/xnnpack/compute.h
+++ b/src/xnnpack/compute.h
@@ -14,7 +14,7 @@
 #include "xnnpack/config-types.h"
 #include "xnnpack/microfnptr.h"
 #include "xnnpack/microparams.h"
-#include "pthreadpool.h"
+#include <pthreadpool.h>
 
 enum xnn_parallelization_type {
   xnn_parallelization_type_invalid = 0,
