From d477120c1668c5b461fdddd0a1428ce5ebb9fe89 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 19 Dec 2024 19:27:29 +0800
Subject: [PATCH] Add files via upload

---
 packaging/pep517_backend/_backend.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packaging/pep517_backend/_backend.py b/packaging/pep517_backend/_backend.py
index f3ea98d..9d0ad5c 100644
--- a/packaging/pep517_backend/_backend.py
+++ b/packaging/pep517_backend/_backend.py
@@ -343,7 +343,7 @@ def get_requires_for_build_wheel(
         )
 
     c_ext_build_deps = [] if is_pure_python_build else [
-        'Cython >= 3.0.0b3' if IS_PY3_12_PLUS  # Only Cython 3+ is compatible
+        'Cython' if IS_PY3_12_PLUS  # Only Cython 3+ is compatible
         else 'Cython',
     ]
 
