From a69d8d93cc663bd4473ef6155528d92f94213a87 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Sun, 11 Jan 2026 23:06:32 +0800
Subject: [PATCH] Update meson.build

---
 src/glsl/meson.build | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/glsl/meson.build b/src/glsl/meson.build
index c9a85429..1f7199a2 100644
--- a/src/glsl/meson.build
+++ b/src/glsl/meson.build
@@ -30,6 +30,8 @@ if glslang_req.auto() and shaderc.found()
 elif not glslang_req.disabled()
 
   glslang_deps = [
+    cxx.find_library('SPIRV', required: false),
+    cxx.find_library('glslang', required: false),
     cxx.find_library('glslang-default-resource-limits', required: false)
   ]
 
