--- a/libvpl/pkgconfig/vpl.pc.in
+++ b/libvpl/pkgconfig/vpl.pc.in
@@ -1,6 +1,7 @@
-prefix=@pc_rel_prefix@
-libdir=@pc_rel_libdir@
-includedir=@pc_rel_incdir@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
 
 Name: Intel(R) Video Processing Library
 Description: Accelerated video decode, encode, and frame processing capabilities on Intel(R) GPUs
