From 657313ffb31760b6f56ef68f9aecaa34e408d03a Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 14 May 2026 23:58:53 +0800
Subject: [PATCH] Update SConstruct

---
 SConstruct | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SConstruct b/SConstruct
index fd818ece9251d..275bfa9b95439 100644
--- a/SConstruct
+++ b/SConstruct
@@ -3244,7 +3244,7 @@ if not env.TargetOSIs('windows', 'macOS') and (env.ToolchainIs('GCC', 'clang')):
     # CXXFLAGS.
 
     default_targeting_flags_for_architecture = {
-        "aarch64": {"-march=": "armv8.2-a", "-mtune=": "generic"},
+        "aarch64": {"-march=": "armv8-a", "-mtune=": "generic"},
         "i386": {"-march=": "nocona", "-mtune=": "generic"},
         "ppc64le": {"-mcpu=": "power8", "-mtune=": "power8", "-mcmodel=": "medium"},
         "s390x": {"-march=": "z196", "-mtune=": "zEC12"},
