From e218f967185ab30478c56f5677773ba262c79660 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Mon, 11 May 2026 08:02:52 +0800
Subject: [PATCH] Update SConstruct

---
 SConstruct | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/SConstruct b/SConstruct
index dcd591c10d7f4..1867ffc8a1f94 100644
--- a/SConstruct
+++ b/SConstruct
@@ -5070,6 +5070,15 @@ def doConfigure(myenv):
 
     if use_system_version_of_library("snappy"):
         conf.FindSysLibDep("snappy", ["snappy"])
+        
+    if use_system_version_of_library("google-benchmark"):
+        conf.FindSysLibDep("benchmark", ["benchmark"])
+        
+    if use_system_version_of_library("libbson"):
+        conf.FindSysLibDep("libbson", ["bson-1.0"])
+        
+    if use_system_version_of_library("libmongocrypt"):
+        conf.FindSysLibDep("libmongocrypt", ["mongocrypt"])
 
     if use_system_version_of_library("zlib"):
         conf.FindSysLibDep("zlib", ["zdll" if conf.env.TargetOSIs('windows') else "z"])

