--- a/SConstruct
+++ b/SConstruct
@@ -455,6 +455,7 @@ add_option(
 
 # Most of the "use-system-*" options follow a simple form.
 for pack in [
+    ('abseil-cpp',),
     (
         'asio',
         'ASIO',
--- a/src/third_party/SConscript
+++ b/src/third_party/SConscript
@@ -21,13 +21,6 @@ tomcryptSuffix = '-1.18.2'
 variantSuffix = '-1.4.0'
 
 thirdPartyEnvironmentModifications = {
-    'abseil-cpp': {'CPPPATH': ['#/src/third_party/abseil-cpp/dist'], },
-    'cares': {
-        'CPPPATH': [
-            '#src/third_party/cares/dist/include',
-            '#src/third_party/cares/platform/${TARGET_OS}_${TARGET_ARCH}/install/include'
-        ],
-    },
     'folly': {'CPPPATH': ['#src/third_party/folly/dist'], },
     'fmt': {'CPPPATH': ['#src/third_party/fmt/dist/include'], },
     'immer': {'CPPPATH': ['#src/third_party/immer/dist'], },
@@ -118,6 +111,11 @@ if not use_system_version_of_library('boost'):
         'CCFLAGS': ['$NOERROR_FOR_UNKNOWN_WARNING_OPTION_GEN'] if env.ToolchainIs('clang') else []
     }
 
+if not use_system_version_of_library('abseil-cpp'):
+    thirdPartyEnvironmentModifications['abseil-cpp'] = {
+        'CPPPATH' : ['#/src/third_party/abseil-cpp-master/abseil-cpp'],
+    }
+
 if not use_system_version_of_library('snappy'):
     thirdPartyEnvironmentModifications['snappy'] = {
         'CPPPATH': ['#/src/third_party/snappy' + snappySuffix],
@@ -447,17 +445,25 @@ else:
 
 boostEnv.ShimLibrary(name="boost")
 
-abseilDirectory = 'abseil-cpp'
 abseilEnv = env.Clone()
-abseilEnv.InjectThirdParty(libraries=['abseil-cpp'])
-abseilEnv.SConscript(abseilDirectory + '/SConscript', exports={'env': abseilEnv})
-abseilEnv = abseilEnv.Clone(LIBDEPS_INTERFACE=[
-    abseilDirectory + '/absl_city',
-    abseilDirectory + '/absl_hash',
-    abseilDirectory + '/absl_int128',
-    abseilDirectory + '/absl_low_level_hash',
-    abseilDirectory + '/absl_raw_hash_set',
-])
+if use_system_version_of_library("abseil-cpp"):
+    abseilEnv = abseilEnv.Clone(
+        SYSLIBDEPS=[
+            env['LIBDEPS_ABSL_CONTAINER_SYSLIBDEP'],
+            env['LIBDEPS_ABSL_HASH_SYSLIBDEP'],
+            env['LIBDEPS_ABSL_NUMERIC_SYSLIBDEP'],
+        ])
+else:
+    abseilDirectory = 'abseil-cpp-master'
+    abseilEnv = abseilEnv.Clone()
+    abseilEnv.InjectThirdParty(libraries=['abseil-cpp'])
+    abseilEnv.SConscript(abseilDirectory + '/SConscript', exports={ 'env' : abseilEnv })
+    abseilEnv = abseilEnv.Clone(
+        LIBDEPS_INTERFACE=[
+            abseilDirectory + '/absl_container',
+            abseilDirectory + '/absl_hash',
+            abseilDirectory + '/absl_numeric',
+        ])
 
 abseilEnv.ShimLibrary(name="abseil")
 
