diff --git a/SCons/Platform/posix.py b/SCons/Platform/posix.py
index 4c9f8f9ba..d68da1b08 100644
--- a/SCons/Platform/posix.py
+++ b/SCons/Platform/posix.py
@@ -30,0 +31 @@ selection method.
+import os
@@ -82,7 +82,8 @@ def generate(env) -> None:
 
     if 'ENV' not in env:
         env['ENV']        = {}
-    env['ENV']['PATH']    = '/usr/local/bin:/opt/bin:/bin:/usr/bin:/snap/bin'
+    env['ENV']['PATH']    = os.environ["PATH"]
+    env['ENV']['STAGING_DIR']    = '$STAGING_DIR'
     env['OBJPREFIX']      = ''
     env['OBJSUFFIX']      = '.o'
     env['SHOBJPREFIX']    = '$OBJPREFIX'
--- a/SCons/Platform/__init__.py
+++ b/SCons/Platform/__init__.py
@@ -63,6 +63,8 @@ def platform_default():
     care about the machine architecture.
     """
     osname = os.name
+    if 'PLATFORM' in os.environ:
+        return os.environ['PLATFORM']
     if osname == 'java':
         osname = os._osType
     if osname == 'posix':
