--- a/backward.hpp
+++ b/backward.hpp
@@ -213,8 +213,9 @@
 #endif
 #include <signal.h>
 #include <sys/stat.h>
-#include <syscall.h>
+#include <sys/syscall.h>
 #include <unistd.h>
+#include <dlfcn.h>
 
 #if BACKWARD_HAS_BFD == 1
 //              NOTE: defining PACKAGE{,_VERSION} is required before including
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.10)
 project(backward CXX)
 
 # Introduce variables:
