From b5a5a405aef40643a15e0f05e2632bd6cf90186f Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Fri, 1 May 2026 11:45:26 +0800
Subject: [PATCH] Update backward.hpp

---
 backward.hpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/backward.hpp b/backward.hpp
index 263cd1b..a85a315 100644
--- a/backward.hpp
+++ b/backward.hpp
@@ -219,8 +219,7 @@
 #endif
 #include <signal.h>
 #include <sys/stat.h>
-#include <syscall.h>
-#include <unistd.h>
+#include <sys/syscall.h>
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #include <dlfcn.h>
--- a/test_package/CMakeLists.txt
+++ b/test_package/CMakeLists.txt
@@ -1,5 +1,5 @@
 project(backward-package-test)
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.10)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
