From 04b894362f30315ba5fc2dfd664b5a70a67e7005 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Fri, 1 May 2026 10:33:41 +0800
Subject: [PATCH] Update backward.hpp

---
 backward.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backward.hpp b/backward.hpp
index 263cd1b..2e3b5a6 100644
--- a/backward.hpp
+++ b/backward.hpp
@@ -219,8 +219,8 @@
 #endif
 #include <signal.h>
 #include <sys/stat.h>
-#include <syscall.h>
 #include <unistd.h>
+#include <sys/syscall.h>
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #include <dlfcn.h>
--- a/test_package/CMakeLists.txt
+++ b/test_package/CMakeLists.txt
@@ -1,5 +1,5 @@
 project(backward-package-test)
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.10)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup(TARGETS)
