From 60fd870f6a0f0dea3932ba8861cdda74d9db626e Mon Sep 17 00:00:00 2001
From: Mariusz Bialonczyk <manio@skyboo.net>
Date: Sat, 14 Mar 2026 14:48:28 +0100
Subject: [PATCH] ui/live view: add 2x3 and 3x2 layouts

---
 ui/src/Live/Multiview.tsx | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/ui/src/Live/Multiview.tsx b/ui/src/Live/Multiview.tsx
index e01ed38ea..491f97dee 100644
--- a/ui/src/Live/Multiview.tsx
+++ b/ui/src/Live/Multiview.tsx
@@ -24,6 +24,8 @@ const LAYOUTS: Layout[] = [
   { className: "dual", cameras: 2, name: "2" },
   { className: "main-plus-five", cameras: 6, name: "Main + 5" },
   { className: "two-by-two", cameras: 4, name: "2x2" },
+  { className: "two-by-three", cameras: 6, name: "2x3" },
+  { className: "three-by-two", cameras: 6, name: "3x2" },
   { className: "three-by-three", cameras: 9, name: "3x3" },
 ];
 const MAX_CAMERAS = 9;
@@ -254,6 +256,14 @@ const Multiview = (props: MultiviewProps) => {
               gridTemplateColumns: "repeat(2, calc(100% / 2))",
               gridTemplateRows: "repeat(2, calc(100% / 2))",
             },
+            "&.two-by-three": {
+              gridTemplateColumns: "repeat(2, calc(100% / 2))",
+              gridTemplateRows: "repeat(3, calc(100% / 3))",
+            },
+            "&.three-by-two": {
+              gridTemplateColumns: "repeat(3, calc(100% / 3))",
+              gridTemplateRows: "repeat(2, calc(100% / 2))",
+            },
             "&.main-plus-five, &.three-by-three": {
               gridTemplateColumns: "repeat(3, calc(100% / 3))",
               gridTemplateRows: "repeat(3, calc(100% / 3))",
