--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -4122,7 +4122,6 @@ set(SHARK_PUBLIC_HEADERS
 install(FILES ${SHARK_PUBLIC_HEADERS}
 	DESTINATION ${PROJECT_INSTALL_INCLUDEDIR}
 	COMPONENT "Development"
-	EXCLUDE_FROM_ALL
 )
 
 # Install icons and other desktop files for Freedesktop.org-compliant desktops.
@@ -4176,7 +4175,6 @@ endif()
 install(FILES "${CMAKE_BINARY_DIR}/resources/wireshark.pc"
 	DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
 	COMPONENT "Development"
-	EXCLUDE_FROM_ALL
 )
 
 install(
@@ -4250,13 +4248,11 @@ install(
 		${WIRESHARK_INSTALL_CMAKEDIR}
 	COMPONENT
 		"Development"
-	EXCLUDE_FROM_ALL
 )
 
 install(EXPORT WiresharkTargets
 	DESTINATION ${WIRESHARK_INSTALL_CMAKEDIR}
 	COMPONENT "Development"
-	EXCLUDE_FROM_ALL
 )
 
 # This isn't strictly needed but it makes working around debhelper's
