From b2a871a234a125bb4c43787e6a1e809638270f50 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 26 Oct 2025 10:13:05 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 764071a..2340d95 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -3,7 +3,7 @@
 
 [build-system]
 requires = [
-    "setuptools == 78.1.1",
+    "setuptools",
     "wheel",
 ]
 build-backend = "setuptools.build_meta"
