From 5ff9caf0c6ccf3c6bfa0e6ddbe6d7c22c753e4da Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 28 Dec 2025 15:09:21 +0800
Subject: [PATCH] Update Makefile

---
 src/Makefile | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 774930a..12eeae3 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -75,12 +75,12 @@ test:	/tmp/test.wav ../snd/noise.wav always
 	@echo Test complete.
 
 install:	targets
-	-mkdir -p $(INSTALL_PLUGINS_DIR)
-	-mkdir -p $(INSTALL_INCLUDE_DIR)
-	-mkdir -p $(INSTALL_BINARY_DIR)
-	cp ../plugins/* $(INSTALL_PLUGINS_DIR)
-	cp ladspa.h $(INSTALL_INCLUDE_DIR)
-	cp ../bin/* $(INSTALL_BINARY_DIR)
+	-mkdir -p $(PKG_INSTALL_DIR)/usr/lib/ladspa
+	-mkdir -p $(PKG_INSTALL_DIR)//usr/include
+	-mkdir -p $(PKG_INSTALL_DIR)/usr/bin
+	cp ../plugins/* $(PKG_INSTALL_DIR)/usr/lib/ladspa
+	cp ladspa.h $(PKG_INSTALL_DIR)//usr/include
+	cp ../bin/* $(PKG_INSTALL_DIR)/usr/bin
 
 /tmp/test.wav:	targets ../snd/noise.wav
 	../bin/listplugins
