From c83eeffc85445018dcedc7a792302d236557e053 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 25 Dec 2025 23:54:29 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 06f5395c3..645a3755e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -17,6 +17,10 @@ cmake_minimum_required(VERSION 3.22.1)
 project(shaderc)
 enable_testing()
 
+set(CMAKE_CURRENT_BINARY_DIR ".")
+set(spirv-tools_SOURCE_DIR "YourFolderTo-SPIRV-Tools")
+set(glslang_SOURCE_DIR "YourFolderTo-glslang")
+
 if ("${CMAKE_BUILD_TYPE}" STREQUAL "")
   message(STATUS "No build type selected, default to Debug")
   set(CMAKE_BUILD_TYPE "Debug")
@@ -130,7 +134,7 @@ endif(MSVC)
 
 # Configure subdirectories.
 # We depend on these for later projects, so they should come first.
-add_subdirectory(third_party)
+#add_subdirectory(third_party)
 
 add_subdirectory(libshaderc_util)
 add_subdirectory(libshaderc)
