From a372ba48cb99e271b5ff297c51c7597965957999 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sat, 29 Aug 2026 17:57:48 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 695842c971aef..266c1dc206aa3 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -312,10 +312,8 @@ dev = [
     # after which this can be removed as it is already specified in the
     # [build-system] section
     "scikit-build-core>=1.0",
-    "setuptools>=77.0.0,<82",  # 77.0 is the first release with PEP 639 license-files support
+    "setuptools",  # 77.0 is the first release with PEP 639 license-files support
     "spin",
-    "cmake>=3.27",
-    "ninja",
     "numpy",
     "packaging>=24.2",  # required by setuptools>=77 to validate PEP 639 license expressions
     "pyyaml",
