--- a/setup.py
+++ b/setup.py
@@ -129,13 +129,14 @@ def read(*rnames):
         "Framework :: Zope :: 3",
         "Topic :: Software Development :: Libraries :: Python Modules",
     ],
-    packages=find_packages('src'),
+    packages=find_packages('src', exclude=["*.tests", "*.tests.*"]),
     package_dir={'': 'src'},
     namespace_packages=["zope"],
     cmdclass={
         'build_ext': optional_build_ext,
     },
     include_package_data=True,
+    exclude_package_data={'':['tests/*']},
     zip_safe=False,
     install_requires=['setuptools'],
     python_requires='>=3.8',
