From a107f2ea5ef0e8051805fb29ea6c08aa1fb9ec3e Mon Sep 17 00:00:00 2001
From: "Christian W. Zuckschwerdt" <christian@zuckschwerdt.org>
Date: Thu, 9 Oct 2025 17:59:59 +0200
Subject: [PATCH] Update for compat with newer CMake

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 02d3492..e911388 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 ########################################################################
 # Build Soapy SDR support module for blade RF
 ########################################################################
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 2.8.12...3.10)
 project(SoapyBladeRF CXX)
 
 find_package(SoapySDR "0.4" NO_MODULE)
