From 77097b7d5f62779985a8c37e56207367fffe2f78 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Fri, 26 Sep 2025 06:25:10 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 ext/clip/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/clip/CMakeLists.txt b/ext/clip/CMakeLists.txt
index 3624d3e2..644e53ba 100644
--- a/ext/clip/CMakeLists.txt
+++ b/ext/clip/CMakeLists.txt
@@ -64,7 +64,7 @@ else()
   list(APPEND CLIP_SOURCES clip_none.cpp)
 endif()
 
-add_library(clip ${CLIP_SOURCES})
+add_library(clip ${LIBRARY_LINKING_TYPE} ${CLIP_SOURCES})
 
 if(WIN32)
   target_link_libraries(clip shlwapi)
