From 91799e90edda3ec2c2c8ad126f43b7b27b899593 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Fri, 26 Sep 2025 09:10:01 +0800
Subject: [PATCH] Update CMakeLists.txt

---
 lib/wmchat/go/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/wmchat/go/CMakeLists.txt b/lib/wmchat/go/CMakeLists.txt
index 2c5ca032..cfb81570 100644
--- a/lib/wmchat/go/CMakeLists.txt
+++ b/lib/wmchat/go/CMakeLists.txt
@@ -34,7 +34,7 @@ set(LIB "libcgowm${CMAKE_SHARED_LIBRARY_SUFFIX}")
 add_custom_command(OUTPUT ${OUTPUT_DIR}/${LIB}
   DEPENDS ${SRCS}
   WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  COMMAND env GOPATH=${GOPATH} CGO_ENABLED=1 go build -buildmode=${BUILDMODE} ${CUSTOM_GO_FLAGS}
+  COMMAND env GOPATH=${GOPATH} CGO_ENABLED=1 /home/runner/work/mosdns/mosdns/sdk/feeds/packages/lang/golang//golang-build.sh build -v -buildvcs=false -trimpath -ldflags "all=-buildid '1758316778' -linkmode external -extldflags '-L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/usr/lib -L/home/runner/work/mosdns/mosdns/sdk/staging_dir/toolchain-aarch64_generic_gcc-13.3.0_musl/lib -fuse-ld=bfd -Wl,-z,now -Wl,-z,relro'" -buildmode=${BUILDMODE} ${CUSTOM_GO_FLAGS}
   -o "${OUTPUT_DIR}/${LIB}"
   ${CMAKE_GO_FLAGS} ./...
   COMMENT "Building Go library")
