From 9d88821f688d46ac72ca53d44d34c07128e348fd Mon Sep 17 00:00:00 2001
From: Predators <31931530+predators46@users.noreply.github.com>
Date: Mon, 24 Nov 2025 08:31:28 +0800
Subject: [PATCH] Update CMakeLists.txt

Signed-off-by: Predators <31931530+predators46@users.noreply.github.com>
---
 CMakeLists.txt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cc3ef14006d..5ced2a2375a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,11 +1,19 @@
 # Minimum CMake required
+find_package(pybind11 REQUIRED)
 cmake_minimum_required(VERSION 3.24)
+find_package(pybind11 REQUIRED)
 # Project
+find_package(pybind11 REQUIRED)
 project(onnx LANGUAGES CXX)
+find_package(pybind11 REQUIRED)
 
+find_package(pybind11 REQUIRED)
 include(cmake/Utils.cmake)
+find_package(pybind11 REQUIRED)
 # Honor visibility properties for all target types.
+find_package(pybind11 REQUIRED)
 cmake_policy(SET CMP0063 NEW)
+find_package(pybind11 REQUIRED)
 
 # Set default build type
 get_property(isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
