--- a/host/lib/CMakeLists.txt
+++ b/host/lib/CMakeLists.txt
@@ -173,7 +173,16 @@ ENDIF(MSVC)
 # Setup libuhd library
 ########################################################################
 ADD_LIBRARY(uhd SHARED ${libuhd_sources})
-TARGET_LINK_LIBRARIES(uhd ${Boost_LIBRARIES} ${libuhd_libs})
+TARGET_LINK_LIBRARIES(uhd
+    PUBLIC
+    Boost::boost
+    Boost::chrono
+    Boost::date_time
+    Boost::filesystem
+    Boost::serialization
+    Boost::thread
+    ${libuhd_libs}
+)
 SET_TARGET_PROPERTIES(uhd PROPERTIES DEFINE_SYMBOL "UHD_DLL_EXPORTS")
 IF(NOT LIBUHDDEV_PKG)
     SET_TARGET_PROPERTIES(uhd PROPERTIES SOVERSION "${UHD_VERSION_MAJOR}")
--- a/host/include/uhd/utils/byteswap.ipp
+++ b/host/include/uhd/utils/byteswap.ipp
@@ -99,7 +99,7 @@
 /***********************************************************************
  * Define the templated network to/from host conversions
  **********************************************************************/
-#include <boost/detail/endian.hpp>
+#include <boost/predef/other/endian.h>
 
 namespace uhd {
 
