--- a/host/include/uhd/utils/byteswap.ipp
+++ b/host/include/uhd/utils/byteswap.ipp
@@ -99,7 +99,7 @@
 /***********************************************************************
  * Define the templated network to/from host conversions
  **********************************************************************/
-#include <boost/detail/endian.hpp>
+#include <boost/predef/other/endian.h>
 
 namespace uhd {
 
