--- a/firmware/fx2/CMakeLists.txt
+++ b/firmware/fx2/CMakeLists.txt
@@ -15,7 +15,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.6...4.0)
 set(CMAKE_TOOLCHAIN_FILE ${CMAKE_SOURCE_DIR}/config/Toolchain-sdcc.cmake)
 PROJECT(USRP1 C)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/config/")
--- a/firmware/usrp3/CMakeLists.txt
+++ b/firmware/usrp3/CMakeLists.txt
@@ -18,7 +18,7 @@
 ########################################################################
 # setup project and compiler
 ########################################################################
-CMAKE_MINIMUM_REQUIRED(VERSION 2.6)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.6...4.0)
 #force the compiler because the check wont use the special flag below
 INCLUDE(CMakeForceCompiler)
 SET(CMAKE_SYSTEM_NAME Generic)
--- a/images/CMakeLists.txt
+++ b/images/CMakeLists.txt
@@ -23,7 +23,7 @@ MESSAGE(STATUS ${UHD_PATCH_OVERRIDE})
 IF(DEFINED UHD_PATCH_OVERRIDE)
     SET(UHD_VERSION_PATCH_OVERRIDE ${UHD_PATCH_OVERRIDE})
 ENDIF(DEFINED UHD_PATCH_OVERRIDE)
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8...4.0)
 PROJECT(UHD-images NONE)
 LIST(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/../host/cmake/Modules)
 INCLUDE(UHDVersion) #sets version variables (used below)
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -23,7 +23,7 @@
 ########################################################################
 # Project setup
 ########################################################################
-CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.8...4.0)
 PROJECT(UHD CXX C)
 ENABLE_TESTING()
 
