From a7b450c4a51305f17a75a35eb80fa2d921b19a10 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Thu, 21 Nov 2024 23:30:36 +0800
Subject: [PATCH] Update meson.options

---
 meson.options | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.options b/meson.options
index d21cbed..a5c1171 100644
--- a/meson.options
+++ b/meson.options
@@ -5,9 +5,9 @@
 # Qhull. You may use the following options to instead link against a system
 # FreeType/Qhull. As an exception, Matplotlib defaults to the system version of
 # FreeType on AIX.
-option('system-freetype', type: 'boolean', value: false,
+option('system-freetype', type: 'boolean', value: true,
        description: 'Build against system version of FreeType')
-option('system-qhull', type: 'boolean', value: false,
+option('system-qhull', type: 'boolean', value: true,
        description: 'Build against system version of Qhull')
 
 # Some of Matplotlib's components are optional: the MacOSX backend (installed
