--- a/clients/tivi/CMakeLists.txt
+++ b/clients/tivi/CMakeLists.txt
@@ -16,7 +16,7 @@
 #
 # @author Werner Dittmann <Werner.Dittmann@t-online.de>
 
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required(VERSION 3.5...4.0)
 
 # setup the Thread include and lib
 find_package(Threads)
--- a/clients/no_client/CMakeLists.txt
+++ b/clients/no_client/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required(VERSION 3.5...4.0)
 
 # setup the Thread include and lib
 find_package(Threads)
--- a/clients/ccrtp/CMakeLists.txt
+++ b/clients/ccrtp/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.6)
+cmake_minimum_required(VERSION 3.5...4.0)
 
 # setup the Thread include and lib
 find_package(Threads)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-cmake_minimum_required(VERSION 3.0)
+cmake_minimum_required(VERSION 3.5...4.0)
 
 PROJECT(libzrtpcpp)
 
