From 3aef5a9458b02263421320afdfbefce83980f85b Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Sun, 22 Dec 2024 20:51:10 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index 50f630b..c678294 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -34,5 +34,5 @@ pylint = "^2.4.4"
 flake8 = "*"
 
 [build-system]
-requires = ["poetry>=1.0.0b1"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry-core>=1.0.0b1"]
+build-backend = "poetry-core.masonry.api"
