From 58c48236f604c1960d6f2475dc18a914adab9b2e Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Tue, 16 Dec 2025 13:13:10 +0800
Subject: [PATCH] Add files via upload

---
 pyproject.toml | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)
 create mode 100644 pyproject.toml

diff --git a/pyproject.toml b/pyproject.toml
new file mode 100644
index 0000000..818791b
--- /dev/null
+++ b/pyproject.toml
@@ -0,0 +1,21 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name         = "tuya-device-sharing-sdk"
+version      = "0.2.6"
+license      = "MIT"
+license-files = ["LICENSE*"]
+description  = "A Python sdk for Tuya Open API, which provides IoT capabilities, maintained by Tuya official"
+readme       = "README.md"
+authors      = [
+    {name = "Tuya Inc.", email = "developer@tuya.com"}
+]
+requires-python = ">=3.13.0"
+
+[project.urls]
+"Homepage" = "https://github.com/tuya/tuya-device-sharing-sdk"
+
+[tool.setuptools.packages.find]
+include = ["tuya_sharing*"]
