From 964d9c43226e1d44635379e704ac4e060b58db10 Mon Sep 17 00:00:00 2001
From: esaaprillia <esaapriliasalsabila@gmail.com>
Date: Wed, 28 Jan 2026 11:45:08 +0800
Subject: [PATCH] Update pyproject.toml

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 6875006..ce08a89 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -82,7 +82,7 @@ compile = [
 ]
 dist = []
 install = []
-setup = [
+#setup = [
     "--vsenv",  # Force use of MSVC on Windows, ignored on other platforms
 ]
 
